"use strict";(self.webpackChunkmgportalv2=self.webpackChunkmgportalv2||[]).push([[343],{2069:(e,a,t)=>{t.d(a,{Z:()=>s});var n=t(7294);const s=()=>n.createElement("svg",{"aria-hidden":"true",className:"vf-card__heading__icon | vf-icon vf-icon-arrow--inline-end",width:"1em",height:"1em",xmlns:"http://www.w3.org/2000/svg",viewBox:"0 0 24 24",preserveAspectRatio:"none"},n.createElement("path",{d:"M0 12c0 6.627 5.373 12 12 12s12-5.373 12-12S18.627 0 12 0C5.376.008.008 5.376 0 12zm13.707-5.209l4.5 4.5a1 1 0 010 1.414l-4.5 4.5a1 1 0 01-1.414-1.414l2.366-2.367a.25.25 0 00-.177-.424H6a1 1 0 010-2h8.482a.25.25 0 00.177-.427l-2.366-2.368a1 1 0 011.414-1.414z",fill:"currentColor",fillRule:"nonzero"}))},5343:(e,a,t)=>{t.r(a),t.d(a,{default:()=>i});var n=t(7294),s=t(9711),r=t(2531),c=t(2069);const i=()=>n.createElement("section",null,n.createElement("div",{className:"vf-section-header"},n.createElement("h2",{className:"vf-section-header__heading"},"Help"),n.createElement("p",{className:"vf-section-header__text"},"MGnify is a freely available hub for the analysis and exploration of metagenomic, metatranscriptomic, amplicon and assembly data. The resource provides rich functional and taxonomic analyses of user-submitted sequences, as well as analysis of publicly available metagenomic datasets drawn from the European Nucleotide Archive (ENA).")),n.createElement("hr",{className:"vf-divider"}),n.createElement("div",{className:"vf-stack vf-stack--800"},n.createElement("article",{className:"vf-card vf-card--brand vf-card--bordered"},n.createElement("div",{className:"vf-card__content | vf-stack vf-stack--400"},n.createElement("h3",{className:"vf-card__heading"},n.createElement("a",{href:"https://emg-docs.readthedocs.io/en/latest/"},n.createElement("span",{className:"icon icon-generic","data-icon":";"})," User documentation",n.createElement(c.Z,null))),n.createElement("p",{className:"vf-card__text"},"Full user documentation of the MGnify service is located at"," ",n.createElement(r.Z,{id:"doc-link-2",title:"MGnify docs",href:"https://emg-docs.readthedocs.io/en/latest/"},"Read the Docs"),"."),n.createElement("p",{className:"vf-card__text"},"The text is searchable and may be downloaded in a variety of formats. Users may also choose to view the documentation for specific pipeline versions (accessed via the bottom left hand corner of the page). Answers to many of the most commonly asked questions can be found under the"," ",n.createElement(r.Z,{href:"https://emg-docs.readthedocs.io/en/latest/faqs.html"},"FAQ"),".",n.createElement("br",null),"Alternatively you can see a high level"," ",n.createElement(s.rU,{to:"/pipelines"},"view of the pipelines"),"."))),n.createElement("article",{className:"vf-card vf-card--brand vf-card--bordered"},n.createElement("div",{className:"vf-card__content | vf-stack vf-stack--400"},n.createElement("h3",{className:"vf-card__heading"},n.createElement("a",{href:"https://shiny-portal.embl.de/shinyapps/app/06_mgnify-notebook-lab?jlpath=mgnify-examples/home.ipynb"},n.createElement("i",{className:"icon icon-common icon-code"})," Programmatic access and examples",n.createElement(c.Z,null))),n.createElement("p",{className:"vf-card__text"},"MGnify data can be access programmatically using"," ",n.createElement(r.Z,{href:"https://www.ebi.ac.uk/metagenomics/api"}," ","the API"),". The API allows you to use command-line tools, like"," ",n.createElement(r.Z,{href:"https://curl.se"},"curl")," to search for and download data. You can also discover the APIâ€™s endpoints by visiting it in a web browser. The"," ",n.createElement(r.Z,{href:"https://shiny-portal.embl.de/shinyapps/app/06_mgnify-notebook-lab?jlpath=mgnify-examples/home.ipynb"},"MGnify Jupyter Lab")," ","server hosts examples of data analysis using R and Python. These are live examples that you can modify without downloading or installing any software."))),n.createElement("article",{className:"vf-card vf-card--brand vf-card--bordered"},n.createElement("div",{className:"vf-card__content | vf-stack vf-stack--400"},n.createElement("h3",{className:"vf-card__heading"},n.createElement("a",{href:"https://hmmer-web-docs.readthedocs.io/en/latest/index.html"},n.createElement("span",{className:"icon icon-functional","data-icon":"1"})," Sequence search",n.createElement(c.Z,null))),n.createElement("p",{className:"vf-card__text"},"This allows protein sequence searches of representatives from our non-redundant protein database using HMMER. Comprehensive documentation on the HMMER web service, including API, is available at"," ",n.createElement(r.Z,{href:"https://hmmer-web-docs.readthedocs.io/en/latest/index.html"},"HMMER Read the Docs site"),". The options specifically associated with the MGnify sequence search service are provided in the User Documentation above. There is a delay between releasing a new version of the protein database for"," ",n.createElement(r.Z,{id:"ftp-link",title:"MGnify protein database FTP",href:"http://ftp.ebi.ac.uk/pub/databases/metagenomics/peptide_database/current_release"},"download"),", and implementing the new release in the MGnify sequence search. Please ensure you are aware which version you are using."))),n.createElement("article",{className:"vf-card vf-card--brand vf-card--bordered"},n.createElement("div",{className:"vf-card__content | vf-stack vf-stack--400"},n.createElement("h3",{className:"vf-card__heading"},n.createElement("a",{href:"http://ftp.ebi.ac.uk/pub/databases/metagenomics/peptide_database/current_release/README.txt"},n.createElement("span",{className:"icon icon-functional","data-icon":"="})," ","Downloading our protein sequence database",n.createElement(c.Z,null))),n.createElement("p",{className:"vf-card__text"},"The MGnify protein database is now so large, that we can not provide a service to allow a search against the complete collection (currently over 2.4 billion sequences). The cluster representatives, complete database and annotations are"," ",n.createElement(r.Z,{id:"ftp-link",title:"MGnify protein database FTP",href:"http://ftp.ebi.ac.uk/pub/databases/metagenomics/peptide_database/current_release"},"available for download"),". We periodically update this dataset, so please ensure you know the version you are using. Please view the"," ",n.createElement("a",{id:"ftp-readme-link",title:"Protein database FTP readme",href:"http://ftp.ebi.ac.uk/pub/databases/metagenomics/peptide_database/current_release/README.txt"},"README.txt")," ","for the complete list of files and their descriptions."))),n.createElement("article",{className:"vf-card vf-card--brand vf-card--bordered"},n.createElement("div",{className:"vf-card__content | vf-stack vf-stack--400"},n.createElement("h3",{className:"vf-card__heading"},n.createElement("a",{href:"https://www.ebi.ac.uk/training/online/about-train-online"},n.createElement("span",{className:"icon icon-generic","data-icon":"t"})," Training materials",n.createElement(c.Z,null))),n.createElement("div",null,"This provide training materials as part of the"," ",n.createElement(r.Z,{title:"EBI Train online",href:"https://www.ebi.ac.uk/training/online/about-train-online"},"EBI Train online"),":",n.createElement("ul",{className:"training"},n.createElement("li",null,n.createElement(r.Z,{title:"Quick Tour",href:"https://www.ebi.ac.uk/training/online/course/ebi-metagenomics-portal-quick-tour"},"Tutorial about the website")),n.createElement("li",null,n.createElement(r.Z,{title:"Submitting data to ENA",href:"https://www.ebi.ac.uk/training/online/course/ebi-metagenomics-portal-submitting-metagenomics-da"},"Tutorial describing the data submission process")),n.createElement("li",null,n.createElement(r.Z,{title:"Analysis and exploration",href:"https://www.ebi.ac.uk/training/online/course/ebi-metagenomics-analysing-and-exploring-metagenomics-data"},"A webinar explaining about MGnify and the analysis you can perform")),n.createElement("li",null,n.createElement(r.Z,{title:"Metagenomics Bioinformatics course run at EMBL-EBI in 2018",href:"https://www.ebi.ac.uk/training/online/course/metagenomics-bioinformatics"},"Recorded materials from the Metagenomics Bioinformatics course run at EMBL-EBI in 2018")))))),n.createElement("article",{className:"vf-card vf-card--brand vf-card--bordered"},n.createElement("div",{className:"vf-card__content | vf-stack vf-stack--400"},n.createElement("h3",{className:"vf-card__heading"},n.createElement("a",{href:"https://www.ebi.ac.uk/support/metagenomics"},n.createElement("span",{className:"icon icon-generic","data-icon":"s"})," Contact us",n.createElement(c.Z,null))),n.createElement("p",{className:"vf-card__text"},"The"," ",n.createElement(r.Z,{title:"service status page",href:"https://status.mgnify.org"},"service status page")," ","lists any known issues and outages."),n.createElement("p",{className:"vf-card__text"},"If neither the documentation nor our online materials answer your questions, please"," ",n.createElement(r.Z,{title:"EBI's support & feedback form",href:"https://www.ebi.ac.uk/support/metagenomics"},"contact us"),".")))))}}]);
//# sourceMappingURL=343.778af2ad.chunk.js.map