"use strict";(self.webpackChunkmgportalv2=self.webpackChunkmgportalv2||[]).push([[400],{2531:(e,t,l)=>{l.d(t,{Z:()=>a});var n=l(7294);const a=({href:e,id:t,title:l,children:a})=>n.createElement("a",{href:e,title:l,id:t,target:"_blank",rel:"noreferrer noopener",className:"mg-ext"},n.createElement("span",{className:"icon icon-common icon-external-link-alt"})," ",a)},964:(e,t,l)=>{l.d(t,{Z:()=>a});var n=l(6974);const a=()=>{const e=(0,n.TH)();let{pathname:t}=e;t.trim().endsWith("/")&&(t=t.trim().slice(0,-1));const l=t.split("/");return null==l?void 0:l[l.length-1]}},6400:(e,t,l)=>{l.r(t),l.d(t,{default:()=>m});var n=l(7294),a=l(9711),r=l(2531);const c=({onHoverStep:e=(()=>null)})=>n.createElement("table",{className:"pipeline_table"},n.createElement("thead",null,n.createElement("tr",null,n.createElement("th",null,"Â "),n.createElement("th",null,"Tools"),n.createElement("th",null,"Version"),n.createElement("th",null,"Description"),n.createElement("th",null,"How we use it"))),n.createElement("tbody",null,n.createElement("tr",{className:"step0 row-cb",onMouseOver:()=>e(0),onFocus:()=>e(0)},n.createElement("td",null,"1"),n.createElement("td",null,n.createElement(r.Z,{href:"https://github.com/jstjohn/SeqPrep"},"SeqPrep")),n.createElement("td",null,"1.1"),n.createElement("td",null,"A program to merge paired end Illumina reads that are overlapping into a single longer read."),n.createElement("td",null,"Paired-end overlapping reads are merged - we do not perform assembly.")),n.createElement("tr",{className:"step1 row-cb",onMouseOver:()=>e(1),onFocus:()=>e(1)},n.createElement("td",null,"2.1"),n.createElement("td",null,n.createElement(r.Z,{href:"http://www.usadellab.org/cms/?page=trimmomatic"},"Trimmomatic")),n.createElement("td",null,"0.32"),n.createElement("td",null,"A flexible read trimming tool."),n.createElement("td",null,"Low quality trimming (low quality ends and sequences with > 10% undetermined nucleotides removed).")),n.createElement("tr",{className:"step1 row-cb",onMouseOver:()=>e(1),onFocus:()=>e(1)},n.createElement("td",null,"2.2"),n.createElement("td",null,n.createElement(r.Z,{href:"http://www.biopython.org/"},"Biopython")),n.createElement("td",null,"1.54"),n.createElement("td",null,"A set of freely available tools for biological computation written in Python."),n.createElement("td",null,"Sequences < 100 nucleotides in length removed.")),n.createElement("tr",{className:"step1 row-cb",onMouseOver:()=>e(1),onFocus:()=>e(1)},n.createElement("td",null,"2.3"),n.createElement("td",null,n.createElement(r.Z,{href:"http://www.drive5.com/uclust/downloads1_1_579.html"},"UCLUST")),n.createElement("td",null,"1.1.579"),n.createElement("td",null,"A high-performance clustering, alignment and search algorithm."),n.createElement("td",null,"Duplicate sequences removed - clustered on 99% identity for LS454 or on 50 nucleotides prefix identity (using pick_otus.py script in Qiime v1.15).")),n.createElement("tr",{className:"step1 row-cb",onMouseOver:()=>e(1),onFocus:()=>e(1)},n.createElement("td",null,"2.4"),n.createElement("td",null,n.createElement(r.Z,{href:"http://www.repeatmasker.org/"},"RepeatMasker")),n.createElement("td",null,"3.2.2"),n.createElement("td",null,"A program that screens DNA sequences for interspersed repeats and low complexity DNA sequences."),n.createElement("td",null,"Repeat masked - removed reads with 50% or more nucleotides masked.")),n.createElement("tr",{className:"step2 row-cb",onMouseOver:()=>e(2),onFocus:()=>e(2)},n.createElement("td",null,"3"),n.createElement("td",null,n.createElement(r.Z,{href:"http://www.ezbiocloud.net/sw/rrnaselector"},"rRNASelector")),n.createElement("td",null,"1.0.0"),n.createElement("td",null,"A computer program for selecting ribosomal RNA encoding sequences from metagenomic and metatranscriptomic shotgun libraries."),n.createElement("td",null,"Prokaryotic rRNA reads are filtered. We use the hidden Markov models to identify rRNA sequences.")),n.createElement("tr",{className:"step3 row-function",onMouseOver:()=>e(3),onFocus:()=>e(3)},n.createElement("td",null,"4"),n.createElement("td",null,n.createElement(r.Z,{href:"http://omics.informatics.indiana.edu/FragGeneScan/"},"FragGeneScan")),n.createElement("td",null,"1.15"),n.createElement("td",null,"An application for finding fragmented genes in short reads."),n.createElement("td",null,"Reads with predicted coding sequences (pCDS) above 60 nucleotides in length.")),n.createElement("tr",{className:"step4 row-function",onMouseOver:()=>e(4),onFocus:()=>e(4)},n.createElement("td",null,"5"),n.createElement("td",null,n.createElement(r.Z,{href:"https://github.com/ebi-pf-team/interproscan/wiki"},"InterProScan")),n.createElement("td",null,"5.0-beta"),n.createElement("td",null,"A sequence analysis application (nucleotide and protein sequences) that combines different protein signature recognition methods into one resource."),n.createElement("td",null,"Matches are generated against predicted CDS, using a sub set of databases (Pfam, TIGRFAM, PRINTS, PROSITE patterns, Gene3d) from InterPro release 31.0. A summary of Gene Ontology (GO) terms derived from InterPro matches to your sample is provided. It is generated using a reduced list of GO terms called GO slim (version"," ",n.createElement(r.Z,{href:"https://www.ebi.ac.uk/metagenomics/geneontology/subsets/goslim_metagenomics_may2012.obo"},"goslim_goa 2012"),").")),n.createElement("tr",{className:"step5 row-taxon",onMouseOver:()=>e(5),onFocus:()=>e(5)},n.createElement("td",null,"6"),n.createElement("td",null,n.createElement(r.Z,{href:"http://qiime.org/"},"QIIME")),n.createElement("td",null,"1.5.0"),n.createElement("td",null,"An open-source bioinformatics pipeline for performing taxonomic analysis from raw DNA sequencing data."),n.createElement("td",null,"16s rRNA are annotated using the Greengenes reference database (default de novo OTU picking protocol with Greengenes 12.10 reference with reverse strand matching enabled).")))),s={1:{Chart:()=>n.createElement("div",{className:"block_wrapper"},n.createElement("div",{className:"block_container"},n.createElement("div",{className:"mainbranch"},n.createElement("div",{className:"block-lb"},"Raw reads"),n.createElement("div",{className:"arrow_pip "}),n.createElement("div",{className:"block small step0"},"SeqPrep"),n.createElement("div",{className:"arrow_pip "}),n.createElement("div",{className:"block-lb"},"Initial reads"),n.createElement("div",{className:"arrow_pip"}),n.createElement("div",{className:"block step1"},"QC",n.createElement("div",{className:"qclist"},n.createElement("ul",null,n.createElement("li",null,"Trim low quality (Trimmomatic)"),n.createElement("li",null,"Length filtering (Biopython)"),n.createElement("li",null,"Duplicate Removal (UCLUST & Prefix)"),n.createElement("li",null,"Filtering low complexity region (RepeatMasker)")))),n.createElement("div",{className:"arrow_pip"}),n.createElement("div",{className:"block-lb"},"Processed reads"),n.createElement("div",{className:"arrow_pip"}),n.createElement("div",{className:"block step2"},"rRNASelector")),n.createElement("div",{className:"branch"},n.createElement("div",{className:"branch1"},n.createElement("div",{className:"arrow_pip rotate_f"}),n.createElement("div",{className:"block-lb"},"Reads without rRNA"),n.createElement("div",{className:"arrow_pip"}),n.createElement("div",{className:"block step3 function"},"FragGeneScan"),n.createElement("div",{className:"arrow_pip"}),n.createElement("div",{className:"block-lb"},"Predicted CDS"),n.createElement("div",{className:"arrow_pip"}),n.createElement("div",{className:"block step4 function"},"InterProScan"),n.createElement("div",{className:"block-nt"},"Functional analysis")),n.createElement("div",{className:"branch2"},n.createElement("div",{className:"arrow_pip rotate_t"}),n.createElement("div",{className:"block-lb"},"Reads with rRNA"),n.createElement("div",{className:"arrow_pip"}),n.createElement("div",{className:"block-lb"},"16s rRNA"),n.createElement("div",{className:"arrow_pip"}),n.createElement("div",{className:"block step5 taxon"},"QIIME"),n.createElement("div",{className:"block-nt"},"Taxonomic analysis"))))),Table:c}},i=({version:e})=>{const[t,l]=(0,n.useState)(-1);if(!s[Number(e)])return null;const{Chart:a}=s[Number(e)];return n.createElement("section",{className:`vf-content highlight-step-${t}`},n.createElement("h2",null,"Pipeline ",e),n.createElement(a,null),n.createElement("div",{onMouseLeave:()=>l(-1)},n.createElement(c,{onHoverStep:l})))};var o=l(964);const m=()=>{const e=(0,o.Z)();return"pipelines"!==e.toLowerCase()?n.createElement(i,{version:e}):n.createElement("section",{className:"vf-content"},n.createElement("h2",null,"Pipeline release archive"),n.createElement("p",null,"You will find here the list of pipelines that were used to run the analyses."),n.createElement("h3",null,n.createElement("span",null,n.createElement(a.rU,{to:"/pipelines/5"},"Pipeline v.5.0"))," ","(06-Nov-2019)"),n.createElement("ul",null,n.createElement("li",null,"Major upgrade. This version offers specialised workflows for three different data types: amplicon, raw metagenomic/metatranscriptomic reads, and assembly. Each workflow is defined in common workflow language (CWL) and available in the"," ",n.createElement(r.Z,{href:"https://github.com/EBI-Metagenomics/pipeline-v5"},"MGnify v5.0 CWL repository"),". All databases are available from an"," ",n.createElement(r.Z,{href:"https://ftp.ebi.ac.uk/pub/databases/metagenomics/pipeline-5.0/ref-dbs"},"FTP link"),".")),n.createElement("h3",null,n.createElement("span",null,n.createElement(a.rU,{to:"/pipelines/4.1"},"Pipeline v.4.1"))," ","(17-Jan-2018)"),n.createElement("ul",null,n.createElement("li",null,"Upgraded SeqPrep to v1.2 with increased sequence length parameter to deal with longer reads"),n.createElement("li",null,"Upgraded MAPseq to v1.2.2"),n.createElement("li",null,"Rebuilt taxonomic reference database based on SILVA v132"),n.createElement("li",null,"Taxonomic assignments now also available in HDF5 format"),n.createElement("li",null,"Applied fix to the coding sequence prediction step")),n.createElement("h3",null,n.createElement("span",null,n.createElement(a.rU,{to:"/pipelines/4.0"},"Pipeline v.4.0"))," ","(04-Sep-2017)"),n.createElement("ul",null,n.createElement("li",null,"Updated tools: InterProScan"),n.createElement("li",null,"rRNASelector (used to identify 16S rRNA genes) was replaced with Infernal for SSU and LSU gene identification"),n.createElement("li",null,"The QIIME taxonomic classification component was replaced with MAPseq"),n.createElement("li",null,"The Greengenes reference database was replaced with SILVA SSU / LSU version 128, enabling classification of eukaryotes, remapped to a 8-level taxonomy"),n.createElement("li",null,"Prodigal was added to run alongside FragGeneScan as part of a combined gene caller when processing assembled sequences")),n.createElement("h3",null,n.createElement("span",null,n.createElement(a.rU,{to:"/pipelines/3.0"},"Pipeline v.3.0"))," ","(30-Jun-2016)"),n.createElement("ul",null,n.createElement("li",null,"Updated tools: InterProScan, FraGeneScan, QIIME and Trimmomatic"),n.createElement("li",null,"Updated GO slim, based on the analysis of over 22 billion (22x10^9) billion functional annotations"),n.createElement("li",null,"Added identification and masking of transfer RNA genes"),n.createElement("li",null,"Improved quality control statistics (sequence length summary, GC and nucleotide distribution)")),n.createElement("h3",null,n.createElement("span",null,n.createElement(a.rU,{to:"/pipelines/2.0"},"Pipeline v.2.0"))," ","(15-Feb-2015)"),n.createElement("ul",null,n.createElement("li",null,"Removed clustering step during the Quality control (QC)"),n.createElement("li",null,"Added step to mask rRNA on reads (instead of removing reads with rRNA)"),n.createElement("li",null,"Improved performance and tools update")),n.createElement("h3",null,n.createElement("span",null,n.createElement(a.rU,{to:"/pipelines/1.0"},"Pipeline v.1.0"))," ","(09-Dec-2009)"))}}}]);
//# sourceMappingURL=400.2e916918.chunk.js.map