"use strict";(self.webpackChunkmgportalv2=self.webpackChunkmgportalv2||[]).push([[778],{6341:(e,t,a)=>{a.d(t,{p0:()=>l,Sf:()=>o,ZP:()=>s});var n=a(7294),r=a(4018);const i=JSON.parse('[{"title":"MGnify: the microbiome analysis resource in 2020","journal":"Nucleic Acids Research","year":2019,"link":"https://academic.oup.com/nar/article/48/D1/D570/5614179","doi":"10.1093/nar/gkz1035","authors":"Mitchell AL, Almeida A, Beracochea M, Boland M, Burgin J, Cochrane G, Crusoe MR, Kale V, Potter SC, Richardson LJ, Sakharova E, Scheremetjew M, Korobeynikov A, Shlemov A, Kunyavskaya O, Lapidus A and Finn RD","recent":true,"main":true},{"title":"A new genomic blueprint of the human gut microbiota","journal":"Nature","year":2019,"link":"https://www.nature.com/articles/s41586-019-0965-1","doi":"10.1038/s41586-019-0965-1","authors":"Almeida A, Mitchell AL, Boland M, Forster SC, Gloor GB, Tarkowska A, Lawley TD and Finn RD","recent":true},{"title":"Human gastrointestinal bacteria genome and culture collection","journal":"Nature Biotechnology","year":2019,"link":"https://www.nature.com/articles/s41587-018-0009-7","doi":"10.1038/s41587-018-0009-7","authors":"Forster SC, Kumar N, Anonye BO, Almeida A, Viciani E, Stares MD, Dunn M, Mkandawire TT, Zhu A, Browne HP, Shao Y, Pike LJ, Louie T, Mitchell AL, Neville BA, Finn RD, Lawley TD","recent":true},{"title":"Benchmarking taxonomic assignments based on 16S rRNA gene profiling of the microbiota from commonly sampled environments","journal":"GigaScience","year":2017,"link":"https://doi.org/10.1093/gigascience/giy054","doi":"10.1093/gigascience/giy054","authors":"Almeida A, Mitchell AL, Tarkowska A, Finn RD.","recent":true},{"title":"EBI Metagenomics in 2017: enriching the analysis of microbial communities, from sequence reads to assemblies","journal":"Nucleic Acids Research","year":2017,"link":"https://doi.org/10.1093/nar/gkx967","doi":"10.1093/nar/gkx967","authors":"Mitchell AL, Scheremetjew M, Denise H, Potter S, Tarkowska A, Qureshi M, Salazar GA, Pesseat S, Boland M, Hunter FMI, ten Hoopen P, Alako B, Amid C, Wilkinson DJ, Curtis TP, Cochrane G and Finn RD"},{"title":"EBI metagenomics in 2016 - an expanding and evolving resource for the analysis and archiving of metagenomic data","journal":"Nucleic Acids Research","year":2015,"link":"http://nar.oxfordjournals.org/content/44/D1/D595.full","doi":"10.1093/nar/gkv1195","authors":"Mitchell AL, Bucchini F, Cochrane G, Denise H, ten Hoopen P, Fraser M, Pesseat S, Potter S, Scheremetjew M, Sterk P and Finn RD"},{"title":"EBI metagenomics - a new resource for the analysis and archiving of metagenomic data","journal":"Nucleic Acids Research","year":2013,"link":"http://dx.doi.org/10.1093/nar/gkt961","doi":"10.1093/nar/gkt961","authors":"Hunter S, Corbett M, Denise H, Fraser M, Gonzalez-Beltran A, Hunter C, Jones P, Leinonen R, McAnulla C, Maguire E, Maslen J, Mitchell AL, Nuka G, Oisel A, Pesseat S, Radhakrishnan R, Rocca-Serra P, Scheremetjew M, Sterk P, Vaughan D, Cochrane G, Field D and Sansone SA"}]'),o=({title:e,journal:t,year:a,link:r,doi:i,authors:o,maxAuthorsLength:l})=>{const s=l&&o.length>l?((e,t)=>{const a=e.slice(0,t).split(",");return a.pop(),a.push(" et al."),a.join(",")})(o,l):o;return n.createElement("section",{className:"mg-pub"},n.createElement("span",{className:"mg-pub-title"},e,".")," ",n.createElement("span",{className:"mg-pub-journal"},t)," (",n.createElement("span",{className:"mg-pub-year"},a),")"," ",n.createElement("span",{className:"mg-pub-doi"},"doi:",n.createElement("a",{title:e,href:r,className:"mg-ext"},i)),n.createElement("div",{className:"mg-pub-authors"},s,"."))},l=()=>{var e;const t=null===(e=i.filter((e=>e.main)))||void 0===e?void 0:e[0];return n.createElement("div",{className:"embl-grid | vf-content"},n.createElement("div",{className:"vf-section-header mg-pub-section-header"},n.createElement("img",{alt:"Cover of the journal",src:r,className:"mg-pub-cover"})),n.createElement("div",null,n.createElement("p",null,"To cite MGnify, please refer to the following publication:"),t&&n.createElement(o,{title:t.title,journal:t.journal,year:t.year,link:t.link,doi:t.doi,authors:t.authors})))},s=()=>{const[e,t]=(0,n.useState)(!1);return n.createElement("section",{className:"mg-pub-section"},i.filter((t=>e||t.recent)).map((e=>n.createElement("article",{key:e.doi},n.createElement(o,{title:e.title,journal:e.journal,year:e.year,link:e.link,doi:e.doi,authors:e.authors})))),n.createElement("div",{className:"mg-right"},n.createElement("button",{type:"button",className:"vf-button vf-button--sm",onClick:()=>t(!e)},e?"Less":"More"," publications")))}},1778:(e,t,a)=>{a.r(t),a.d(t,{default:()=>m});var n=a(7294),r=a(6341),i=a(4766),o=a(498),l=a(6523),s=a(5647),c=a(3339);const m=()=>n.createElement("section",{className:"vf-content"},n.createElement("h2",null,"The MGnify resource"),n.createElement("p",null,"Microbiome research involves the study of all genomes present within a specific environment. The approach can provide unique insights into the complex processes performed by environmental micro-organisms and their relationship to their surroundings, to each other, and, in some cases, to their host."),n.createElement("p",null,"MGnify offers an automated pipeline for the analysis and archiving of microbiome data to help determine the taxonomic diversity and functional & metabolic potential of environmental samples. Users can submit their own data for analysis or freely browse all of the analysed public datasets held within the repository. In addition, users can request analysis of any appropriate dataset within the European Nucleotide Archive (ENA). User-submitted or ENA-derived datasets can also be assembled on request, prior to analysis."),n.createElement("h2",null,"Staying informed"),n.createElement("p",null,"Follow us on Twitter using"," ",n.createElement("a",{href:"https://twitter.com/MGnifyDB"},"@MGnifyDB")),n.createElement("h2",null,"Cite us"),n.createElement(r.p0,null),n.createElement("h2",null,"Latest publications"),n.createElement(r.ZP,null),n.createElement("h2",null,"Funding"),n.createElement("div",{className:"vf-grid"},n.createElement("p",null,"MGnify currently receives funding support from: the European Unionâ€™s Horizon 2020 Research and Innovation programme (817729 and 824087); the Biotechnology and Biological Sciences Research Council (BB/R015228/1, BB/S009043/1 and BB/N018354/1); ELIXIR, the research infrastructure for Life-Science data; and European Molecular Biology Laboratory (EMBL) core funds."),n.createElement("div",{className:"vf-grid vf-grid__col-2 mg-about-logos"},n.createElement("img",{src:i,alt:"embl logo"}),n.createElement("img",{src:o,alt:"bbsrc logo"}),n.createElement("img",{src:l,alt:"excelerate logo"}),n.createElement("img",{src:c,alt:"innovate logo"}),n.createElement("img",{src:s,alt:"holofood logo"}))))},4018:(e,t,a)=>{e.exports=a.p+"static/bef88e0e94ffc23af445.gif"},498:(e,t,a)=>{e.exports=a.p+"static/0cd014b7b193d9b8b688.png"},4766:(e,t,a)=>{e.exports=a.p+"static/fe2dbedc04a66f23324a.png"},6523:(e,t,a)=>{e.exports=a.p+"static/4bd997697cccdd3db707.png"},5647:(e,t,a)=>{e.exports=a.p+"static/46d33a0a2881f0cfe841.png"},3339:(e,t,a)=>{e.exports=a.p+"static/9e1b122655ed98185e71.png"}}]);
//# sourceMappingURL=778.5c71d9f4.chunk.js.map