/*
-- Query: SELECT * FROM PIPELINE_RELEASE_TOOL
LIMIT 0, 50000

-- Date: 2018-02-20 11:13
*/
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (1,1,1.100,'Low quality trimming (low quality ends and sequences with > 10% undetermined nucleotides removed).');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (1,2,3.000,'Reads with predicted coding sequences (pCDS) above 60 nucleotides in length.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (1,3,4.000,'Matches are generated against predicted CDS, using a sub set of databases (Pfam, TIGRFAM, PRINTS, PROSITE patterns, Gene3d) from InterPro release 31.0. A summary of Gene Ontology (GO) terms derived from InterPro matches to your sample is provided. It is generated using a reduced list of GO terms called GO slim (version <a href=\"https://www.ebi.ac.uk/metagenomics/geneontology/subsets/goslim_metagenomics_may2012.obo\" class=\"ext\">goslim_goa 2012</a>).');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (1,4,1.300,'Duplicate sequences removed - clustered on 99% identity for LS454 or on 50 nucleotides prefix identity (using pick_otus.py script in Qiime v1.15).');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (1,5,1.400,'Repeat masked - removed reads with 50% or more nucleotides masked.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (1,6,2.000,'Prokaryotic rRNA reads are filtered. We use the hidden Markov models to identify rRNA sequences.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (1,7,5.000,'16s rRNA are annotated using the Greengenes reference database (default de novo OTU picking protocol with Greengenes 12.10 reference with reverse strand matching enabled).');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (1,8,1.200,'Sequences < 100 nucleotides in length removed.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (2,1,1.100,'Low quality trimming (low quality ends and sequences with > 10% undetermined nucleotides removed).');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (2,2,3.000,'Reads with predicted coding sequences (pCDS) above 60 nucleotides in length.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (2,9,2.000,'Prokaryotic rRNA reads are filtered. We use the hidden Markov models to identify rRNA sequences.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (2,10,5.000,'16s rRNA are annotated using the Greengenes reference database (default closed-reference OTU picking protocol with Greengenes 13.8 reference with reverse strand matching enabled).');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (2,11,1.200,'Sequences < 100 nucleotides in length removed.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (2,12,4.000,'Matches are generated against predicted CDS, using a sub set of databases (Pfam, TIGRFAM, PRINTS, PROSITE patterns, Gene3d) from InterPro release 31.0. A summary of Gene Ontology (GO) terms derived from InterPro matches to your sample is provided. It is generated using a reduced list of GO terms called GO slim (version <a href=\"https://www.ebi.ac.uk/metagenomics/geneontology/subsets/goslim_metagenomics_may2012.obo\" class=\"ext\">goslim_goa 2012</a>).');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (3,11,1.200,'Sequences < 100 nucleotides in length removed.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (3,13,5.000,'16s rRNA are annotated using the Greengenes reference database (default closed-reference OTU picking protocol with Greengenes 13.8 reference with reverse strand matching enabled).');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (3,14,4.000,'Matches are generated against predicted CDS, using a sub set of databases (Pfam, TIGRFAM, PRINTS, PROSITE patterns, Gene3d) from InterPro release 58.0. A summary of Gene Ontology (GO) terms derived from InterPro matches to your sample is provided. It is generated using a reduced list of GO terms called GO slim (version <a href=\"http://www.geneontology.org/ontology/subsets/goslim_metagenomics.obo\" class=\"ext\">goslim_goa</a>).');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (3,15,1.100,'Low quality trimming (low quality ends and sequences with > 10% undetermined nucleotides removed).');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (3,16,3.000,'Reads with predicted coding sequences (pCDS) above 60 nucleotides in length.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (3,17,0.000,'Paired-end overlapping reads are merged - we do not perform assembly.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (3,18,2.000,'Identification and masking of ncRNAs.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (4,11,1.200,'Sequences < 100 nucleotides in length removed.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (4,15,1.100,'Low quality trimming (low quality ends and sequences with > 10% undetermined nucleotides removed). Adapter sequences removed using Biopython SeqIO package.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (4,16,3.100,'Run as a combined gene caller component, giving priority to Prodigal predictions in the case of assembled sequences or FragGeneScan for short reads (all predictions from the higher priority caller are used, supplemented by any non-overlapping regions predicted by the other).');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (4,17,0.000,'Paired-end overlapping reads are merged - if you want your data assembled, email us.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (4,19,4.000,'Matches are generated against predicted CDS, using a sub set of databases (Pfam, TIGRFAM, PRINTS, PROSITE patterns, Gene3d) from InterPro release 64.0. A summary of Gene Ontology (GO) terms derived from InterPro matches to your sample is provided. It is generated using a reduced list of GO terms called GO slim (version <a href=\"http://www.geneontology.org/ontology/subsets/goslim_metagenomics.obo\" class=\"ext\">goslim_goa</a>).');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (4,20,3.200,'');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (4,21,2.100,'Identification of ncRNAs.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (4,22,5.000,'SSU and LSU rRNA are annotated using SILVAs SSU/LSU version 128 reference database, enabling classification of eukaryotes, remapped to a 8-level taxonomy.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (4,23,2.200,'Removes lower scoring overlaps from cmsearch --tblout files.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (5,11,1.200,'Sequences < 100 nucleotides in length removed.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (5,15,1.100,'Low quality trimming (low quality ends and sequences with > 10% undetermined nucleotides removed). Adapter sequences removed using Biopython SeqIO package.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (5,16,3.100,'Run as a combined gene caller component, giving priority to Prodigal predictions in the case of assembled sequences or FragGeneScan for short reads (all predictions from the higher priority caller are used, supplemented by any non-overlapping regions predicted by the other).');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (5,19,4.000,'Matches are generated against predicted CDS, using a sub set of databases (Pfam, TIGRFAM, PRINTS, PROSITE patterns, Gene3d) from InterPro release 64.0. A summary of Gene Ontology (GO) terms derived from InterPro matches to your sample is provided. It is generated using a reduced list of GO terms called GO slim (version <a href=\"http://www.geneontology.org/ontology/subsets/goslim_metagenomics.obo\" class=\"ext\">goslim_goa</a>).');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (5,20,3.200,'');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (5,21,2.100,'Identification of ncRNAs.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (5,23,2.200,'Removes lower scoring overlaps from cmsearch --tblout files.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (5,24,0.000,'Paired-end overlapping reads are merged - if you want your data assembled, email us.');
INSERT INTO `PIPELINE_RELEASE_TOOL` (`PIPELINE_ID`,`TOOL_ID`,`TOOL_GROUP_ID`,`HOW_TOOL_USED_DESC`) VALUES (5,25,5.000,'SSU and LSU rRNA are annotated using SILVAs SSU/LSU version 132 reference database, enabling classification of eukaryotes, remapped to a 8-level taxonomy.');
