/*
-- Query: SELECT * FROM STUDY WHERE EXT_STUDY_ID IN ('SRP001743','ERP000118','SRP000319','SRP000240','ERP000554','SRP000110','SRP000664','SRP002017','SRP003198','SRP002423','SRP002437','SRP001803','MetaSoil','SRP001095','ERP011824','ERP011843','ERP006800','ERP010956','ERP011237','ERP011248','ERP011822','ERP011407','ERP011266','ERP011856','ERP012163','ERP011921','SRP031871','ERP012642','ERP012647','ERP012815','ERP012221','ERP012766','ERP012888','ERP012964','ERP013028','ERP013031','ERP013033','SRP022254','ERP012972','ERP012953','ERP020679','ERP004413','ERP009442','ERP015373','SRP095053','DRP001287','SRP059258','ERP009703')
LIMIT 0, 50000

-- Date: 2018-02-20 11:06
*/
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (256,'Joint Genome Institute, U.S. Department of Energy','none',1,NULL,'Many natural ecosystems are efficiently degrading lignocellulose and harbor enzymes that could potentially be harnessed for the production of next generation biofuels from plant biomass. We have used shotgun metagenome sequencing to provide a survey of the microbial community and potential metabolic pathways present in the tropical rain forest soil of Puerto Rico. Wet tropical forest soils are some of the most productive and diverse terrestrial ecosystems on earth. These soils are characterized by high iron concentrations and regular fluctuations between oxic and anoxic conditions on a scale of days to weeks. These dynamic conditions suggest that bacterial and archaeal processes are more important in this type of environment than in other ecosystems.','SRP001743','Rainforest Soil','IN_PROGRESS','HARVESTED','not available','not available','2016-01-20 14:12:06',NULL,250,'2011/01/SRP001743','2010-02-26 00:00:00','PRJNA169357');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (257,'Plymouth Marine Laboratory','time',1,NULL,'Here we present a multi-omic study of the Bacterial and Archaeal diversity found at the L4 long-term marine observatory.  We have previously generated a six year time series from L4 that showed that the bacterial community shows strong seasonal structuring and diversity peaks every year on the winter solstice.  Here we further confirm this pattern by extending this study to include genes and transcripts generated for eight additional time points in 2008.  This data includes further 16S datasets as well as eight metagenomes (1.2 GB) and eight metatranscriptomes (157MB).  These time points cover three seasons (Jan, April and August) and include day and night (diel) publicSamples.  In addition, the August publicSamples (4) include 4 consecutive samplings within 24 hours at six hour intervals.  Using these data we test whether Archaea also show the same observed seasonal patterns and whether these patterns hold true at the gene (functional) level. Analysis of this combined data set allows five main conclusions to be drawn.  First, Archaea show evidence of following the same seasonal patterns as Bacteria, but have ~6% the richness.  Second, for both Bacteria and Archaea, we confirm that higher 16S diversity reflects higher diversity at the gene-level (including expressed genes) and this diversity also peaks at the winter solstice. Third, interestingly, detectable diversity appears to be higher at night, and this is of special potential relevance as there is more diversity in winter when nights are longer.  Fourth, despite the diversity of these communities, night and day publicSamples taken using Lagrangian (drift) sampling were successful in isolating the same community suggesting communities are more structured than is commonly believed. Finally, this data, as expected, contains a large proportion of orphan genes without known homologues.  When compared to the housekeeping genes identified through SEED subsystem classification, these unknown genes appear to be driving the differences between publicSamples across seasons.  This underscores the importance of determining the functions of more of these sequences in the future. In summary, this study further confirms the strong seasonal patterns characterizing both the bacterial and archaeal communities at this important marine site.  The finding that, despite the huge diversity of these communities, there are evident signs of predictable patterns and detectable stability over time provides compelling evidence that renewed efforts should be focused on finding deterministic patterns in even the most complex microbial communities. The Western Channel Observatory maintains more than 100 years of continual environmental monitoring, including oceanographic and biological measures. At PML we are committed to developing this outstanding time series to include the most up to date techniques available. Our aim is to provide an holistic ecologically relevant dataset that can be used to inform environmental models and help with prediction of the impact that climate change will have on the coastal seas of the United Kingdom. With further development and comparative publicStudies this information can help to inform on global marine environments. To this end we have been employing the latest in high-throughput sequencing technology to investigate the microbial diversity and community function in this ecosystem. The technologies applied include 16S rDNA tag-pyrosequencing to deep sequence microbial diversity, as well as total community metagenomics and metatranscriptomics. This initial study includes 16S rDNA tag sequencing data for bacteria and archaeal groups as well as metagenomics and metatranscriptomics all performed on the GS-flx titanium platform using techniques outlined in Gilbert et al (2009 – Environmental Microbiology) and Gilbert et al (2008 – PloS One.) We will continue to add to this repository to maintain the temporal analysis of this community.','ERP000118','Western English Channel diurnal study','IN_PROGRESS','HARVESTED','not available','Jack Gilbert','2016-01-20 14:12:06',NULL,158,'2010/02/ERP000118','2010-03-22 00:00:00','PRJEB2064');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (258,'CGS-GL','obese vs lean',1,NULL,'We have characterized the fecal microbial communities of adult female monozygotic and dizygotic twin pairs concordant for leanness or obesity, and their mothers. The results demonstrate that a diversity of organismal assemblages can nonetheless yield a core microbiome at a functional level, and that deviations from this core are associated with different physiologic states (obese versus lean)','SRP000319','A core gut microbiome in obese and lean twins','FINISHED','HARVESTED','not available','Turnbaugh PJ','2016-01-20 14:12:06',NULL,361,'2010/03/SRP000319','2010-02-26 00:00:00','PRJNA32089');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (259,'G2L (Goettingen Genomics Lab)','none',1,NULL,'The largest part of the Earth\'s microbial biomass is stored in cold environments, which represent almost untapped reservoirs of novel species, processes, and genes. In this study, the first metagenomic survey of the metabolic potential and phylogenetic diversity of a microbial assemblage present in glacial ice is presented. DNA was isolated from glacial ice of the Northern Schneeferner, Germany. Pyrosequencing of this DNA yielded 1,076,539 reads (239.7 Mbp). The phylogenetic composition of the prokaryotic community was assessed by evaluation of a pyrosequencing-derived data set and sequencing of 16S rRNA genes. The Proteobacteria (mainly Betaproteobacteria), Bacteroidetes, and Actinobacteria  were the predominant phylogenetic groups. In addition, isolation  of psychrophilic microorganisms was performed, and 13 different  bacterial isolates were recovered. Analysis of the 16S rRNA gene sequences of the isolates revealed that all were affiliated to the predominant groups. As expected for microorganisms residing in a low-nutrient environment, a high metabolic versatility with respect to degradation of organic substrates was detected by analysis of the pyrosequencing-derived data set. The presence of autotrophic microorganisms was indicated by identification of genes typical for different ways of carbon fixation. In accordance  with the results of the phylogenetic samples, in which mainly  aerobic and facultative aerobic bacteria were detected, genes  typical for central metabolism of aerobes were found. Nevertheless, the capability of growth under anaerobic conditions was indicated by genes involved in dissimilatory nitrate/nitrite reduction. Numerous characteristics for metabolic adaptations associated with a psychrophilic lifestyle, such as formation of cryoprotectants and maintenance of membrane fluidity by the incorporation of unsaturated fatty acids, were detected. Thus, analysis of the glacial metagenome provided insights into the microbial life in frozen habitats on Earth, thereby possibly shedding light onto microbial life in analogous extraterrestrial environments.','SRP000240','Glacier Metagenome','IN_PROGRESS','HARVESTED','not available','Rolf Daniel','2016-01-20 14:12:06',NULL,103,'2010/04/SRP000240','2010-02-26 00:00:00','PRJNA30701');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (260,'JIC',NULL,1,'2011-07-15','The interaction between microbes and plants in the region around plant roots (rhizosphere) is a key determinant of plant productivity. Here we use meta-transcriptomic sequences of mRNA extracted from 4 pooled 1g wheat rhizosphere soil samples to map metabolic interactions in the rhizosphere.','ERP000554','Wheat Rhizosphere, John Innes Centre','FINISHED','SUBMITTED','philip.poole@plants.ox.ac.uk','Philip Poole','2016-01-20 14:12:06','Webin-804',220,'2014/10/ERP000554','2011-03-16 00:00:00','PRJEB2451');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (264,'Center for Comparative Genomics and Bioinformatics, PennState University, USA','none',5,NULL,'Metagenomics is expanding our knowledge of the gene content, functional significance, and genetic variability in natural microbial communities. Still, there exists limited information concerning the regulation and dynamics of genes in the environment. We report here global analysis of expressed genes in a naturally occurring microbial community. We first adapted RNA amplification technologie s to produce large amounts of cDNA from small quantities of total microbial community RNA. The fidelity of the RNA amplification pro cedure was validated with Prochlorococcus cultures, and then applied to a microbial assemblage collected in the oligotrophic Pacific Ocean. Microbial community cDNAs were analyzed by pyrosequencing, and compared to microbial community genomic DNA sequences determi ned from the same sample. Pyrosequencing-based estimates of microbial community gene expression compared favorably to independent as sessments of individual gene expression using quantitative PCR. Genes associated with key metabolic pathways in open ocean microbial species, including genes involved in photosynthesis, carbon fixation, and nitrogen acquisition, and a number of genes encoding hypo thetical proteins, were highly represented in the cDNA pool. Genes present in the variable regions of Prochlorococcus genomes were a mong the most highly expressed, suggesting these encode proteins central to cellular processes in specific genotypes. Although many transcripts detected were highly similar to genes previously detected in ocean metagenomic surveys, a significant fraction (~ 50%) w ere unique. Thus, microbial community transcriptomic analyses revealed not only indigenous gene- and taxon-specific expression patte rns, but also new gene categories, undetected in previous DNA-based metagenomic surveys.','SRP000110','HOT station, Central North Pacific Gyre, ALOHA','FINISHED','HARVESTED','not available','not available','2016-01-20 14:12:06',NULL,158,'2011/05/SRP000110','2010-02-26 00:00:00','PRJNA16339');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (274,'THE PENNSYLVANIA STATE UNIVERSITY','journey location',1,NULL,'How many species inhabit our immediate surrounding? A straightforward collection technique suitable for answering this question is known to anyone who has ever driven a car at highway speeds. The windshield of a moving vehicle is subjected to numerous insect strikes and can be used as a collection device for representative sampling. Combined with next-generation sequencing this approach allows for species identification. Here we describe an application of this methodology to two geographic locations within the Northeastern United States. We identified 61 arthropod and 39 plant species, seven of which showed significant difference in the relative abundance between the two localities. Importantly, we developed a novel robust computational framework for the comparison of relative species abundances between geographic locations, which can be used in other read-count-based applications of next-generation sequencing, such as comparison of expression levels. \r\nThe main aim of our study was to contrast eukaryotic species differences between two distinct geographical location in northeastern United States. DNA, isolated from the material stuck to the tapped front bumper of a car driven in these locations, was sequenced using 454 (Roche) FLX pyrosequencing technology.','SRP000664','Windshield splatter','FINISHED','HARVESTED','not available','not available','2016-01-20 14:12:06',NULL,82,'2011/05/SRP000664','2010-02-26 00:00:00',NULL);
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (275,'DIVISIÓN DE MICROBIO, Evolutionary Genomics Group, Universidad Miguel Hernández, Alicante, Spain.','none',1,NULL,'The Deep Chlorophyll Maximum (DCM) is a zone of maximal photosynthetic activity, generally located towards the base of the photic zone in lakes and oceans. The DCM in the Mediterranean is a seasonal phenomenon. The metagenome from a single sample of a mature Mediterranean DCM community has been 454 pyrosequenced both directly and after cloning in fosmids. The comparison between the two approaches revealed a bias in the fosmid libraries against low GC DNA and specifically against the two most dominant members of the community, Candidatus Pelagibacter and Prochlorococcus marinus subsp. pastoris, thus unexpectedly providing a feasible method to obtain large genomic fragments from other less prevalent members of this community. This study is the first to be carried out at this sequencing depth (ca. 600 Mbp combining direct and fosmid sequencing) at any DCM. Our results indicate a microbial community massively dominated by the high-light adapted P. marinus subsp. pastoris, Synechococcus sp. and the heterotroph Candidatus Pelagibacter. The sequences retrieved were remarkably similar to the existing genome of P. marinus subsp. pastoris with a nucleotide identity over 98%. Besides, we found a large number of cyanophages that could prey on this microbe although sequence conservation was much lower. The high abundance of phage sequences in the cellular size fraction indicated a remarkably high proportion of cells suffering phage lytic attack. In addition, several fosmids clearly belonging to Group II Euryarchaeota were retrieved and recruited many fragments from the total DNA sequencing suggesting that this group might be quite abundant in this habitat.\r\nWe have extracted metagenomic DNA from the pico-planktonic 5-0.2 micrometer size fraction (mostly prokaryotic cells) from a single sample obtained in mid October at 50 m depth and analyzed the DNA from this sample (without cloning or amplification) by 454 pyrosequencing (hereafter referred to as Direct Sequencing, DS). We also used 454 to sequence ca. 1152 randomly selected fosmid clones from a metagenomic fosmid library constructed from the same DNA sample. Here we compare the output from both methods and the picture of the microbial community that they provide.','SRP002017','The Metagenome of the deep chlorophyll maximum in the Mediterranean studied by direct and fosmid library 454 pyrosequencing',NULL,'HARVESTED','not available','not available','2016-01-20 14:12:06',NULL,158,'2013/09/SRP002017','2010-05-05 00:00:00',NULL);
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (276,'Department of Life Science, Chung-ang university','fermentation time',1,'2013-01-26','Kimchi, the Korean culture\'s traditional food, is made from vegetables by fermentation. In this study, metagenomic approaches were used to monitor changes in bacterial populations, metabolic potential, and overall genetic features of the microbial community during the 29-day fermentation. Metagenomic DNA was extracted from kimchi samples obtained periodically and followed by sequencing using a 454 GS-FLX Titanium system, which yielded a total of 701,556 reads, with an average read length of 438 bp. Phylogenetic analysis based on 16S rRNA genes from the metagenome indicated that the kimchi microbiome was dominated by members of three genera; Leuconostoc, Lactobacillus, and Weissella. Assignment of metagenomic sequences to SEED categories of MG-RAST revealed a genetic profile characteristic of heterotrophic lactic acid fermentation of carbohydrates, which was supported by the detection of mannitol, lactate, acetate, and ethanol as fermentation products. When the metagenomic reads were mapped onto the database of completed genomes, the Leuconostoc mesenteroides subsp. mesenteroides ATCC 8293 and Lactobacillus sakei subsp. sakei 23K genomes were highly represented. These same two genera were confirmed to be important in kimchi fermentation when the majority of kimchi metagenomic sequences showed very high identity to Leuconostoc mesenteroides and Lactobacillus genes. Besides microbial genome sequences, a surprisingly large number of phage DNA sequences were identified from the cellular fractions, possibly indicating that a high proportion of cells were infected by bacteriophage during fermentation. Overall, these results provide insights into the kimchi microbial community and also shed light on fermentation processes carried out broadly by complex microbial communities.\r\nWe have extracted metagenomic DNA from fermented food kimchi and sequenced the DNA from this sample by 454 GS-FLX titanium without cloning or amplification. Here we analyze changes in bacterial populations, metabolic potential, and overall genetic features of the microbial community during the 29-day fermentation using metagenomic approaches.','SRP003198','Kimchi metagenome','FINISHED','HARVESTED','not available','not available','2016-01-20 14:12:06','Webin-29314',35,'2014/02/SRP003198','2010-10-27 00:00:00','PRJNA50815');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (277,'NIDDK','Crohn\'s Disease',1,NULL,'Inflammatory bowel diseases (IBD), such as Crohn’s disease, are chronic, immunologically mediated disorders that have severe medical consequences. The current hypothesis is that these diseases are due to an overly aggressive immune response to a subset of commensal enteric bacteria. Studies to date on IBD have suggested that the disorder may be caused by a combination of bacteria and host susceptibility; however the etiologies of these diseases remain an enigma. \r\nIn this application, we propose to develop and demonstrate the ability to profile Crohn’s disease at an unprecedented molecular level by elucidation of specific biomarkers (bacterial strains, genes, or proteins) that correlate to disease symptoms. To achieve this goal, we will employ a multidisciplinary approach based on metagenomic and metaproteomic molecular tools to elucidate the composition of the commensal microbiota in monozygotic twins that are either healthy or exhibit Crohn’s disease (for concordant, both are diseased; for discordant, one is healthy and one is diseased). \r\nThe central hypotheses of this proposal are (1) that specific members and/or functional activities of the gastrointestinal (GI) microbiota differ in patients with Crohn’s disease as compared to healthy individuals, and (2) that it will be possible to elucidate microbial signatures which correlate with the occurrence and progression of this disease by integration of data obtained from 16S rRNA-based molecular fingerprinting, metagenomics, and metaproteomics approaches. To address these hypotheses, three specific aims are proposed: 1) Obtain data on community gene content (metagenome) in a subset of healthy twins and twins with Crohn’s Disease to assess potential differences in the metabolic capabilities of the gut microbiota associated with CD, 2) Obtain data on community protein content (metaproteome) in a subset of healthy twins and twins with Crohn’s Disease to assess the state of expressed proteins associated with CD, 3) Apply various statistical clustering and classification methods to correlate/associate microbial community composition, gene and protein content with patient metadata, including metabolite profiles and clinical phenotype. \r\nThe ultimate goal of these efforts is to identify novel biomarkers for non-invasive diagnostics of CD and to eventually identify drug targets (i.e. bacterial strains) for cure or suppression of disease symptoms. \r\nPUBLIC HEALTH RELEVANCE: This study aims to unravel the contribution of the bacteria that normally inhabit the human gastrointestinal tract to Crohn’s disease by using a multidisciplinary approach to study changes in the structure and function of gut microbial communities in three sets of patient cohorts who have Crohn’s disease. These results will be compared with those obtained from the study of healthy individuals and have the potential to identify new biomarkers of disease severity, location, and progression.','SRP002423','Human Gut Microbiome in Crohn\'s Disease',NULL,'HARVESTED','not available','not available','2016-01-20 14:12:06',NULL,361,'2011/05/SRP002423','2010-05-27 00:00:00','PRJNA46321');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (278,'Cornell University','time',1,NULL,'The colonization process of the infant gut microbiome has been called chaotic, but this view could reflect insufficient documentation of the factors affecting the microbiome. We performed a 2.5-year case study of the assembly of the human infant gut microbiome to relate life events to microbiome composition and function. Sixty fecal samples were collected from a healthy infant along with a diary of diet and health status. Analysis of >300,000 16S rRNA genes indicated that the phylogenetic diversity of the microbiome increased gradually over time and that changes in community composition conformed to a smooth temporal gradient. In contrast, major taxonomic groups showed abrupt shifts in abundance corresponding to changes in diet or health. Community assembly was nonrandom: we observed discrete steps of bacterial succession punctuated by life events. Furthermore, analysis of ~500,000 DNA metagenomic reads from 12 fecal samples revealed that the earliest microbiome was enriched in genes facilitating lactate utilization, and that functional genes involved in plant polysaccharide metabolism were present prior to the introduction of solid food, priming the infant gut for an adult diet. However, ingestion of table foods caused a sustained increase in the abundance of Bacteroidetes, elevated fecal short chain fatty acid levels, enrichment of genes associated with carbohydrate utilization, vitamin biosynthesis and xenobiotic degradation, and a more stable community composition, all of which are characteristic of the adult microbiome. This study revealed that seemingly chaotic shifts in the microbiome could be attributed to life events.','SRP002437','Developing infant gut microbiome',NULL,'HARVESTED','not available','Ruth E. Ley','2016-01-20 14:12:06','Webin-29314',361,'2014/02/SRP002437','2010-08-06 00:00:00',NULL);
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (282,'J. Craig Venter Institute',NULL,1,NULL,'Microbial metagenome from a lake in Antarctica.\r\nMicrobial community isolated from an aquatic lake in Antarctica. ','SRP001803','Antarctica Aquatic Microbial Metagenome',NULL,'HARVESTED','not available','not available','2016-01-20 14:12:06','Webin-29314',132,'2014/02/SRP001803','2010-02-26 00:00:00','PRJNA33179');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (288,'CCME','none',1,NULL,'This project describes a population genomic study focused on determining the evolutionary rates in natural populations, and to correlate human disturbance to major evolutionary divergence events. Biofilm samples from acid mine drainage were taken in the period 2006-2010 at four locations in the Richmond Mine (Iron Mt., CA). Community genomic DNA was extracted and sequenced by either 454 or Illumina sequencing.','SRP001095','Brazos-Trinity Basin Sediment Metagenome: IODP Site 1320',NULL,'HARVESTED','jfbiddle@udel.edu','J Biddle','2016-01-20 14:12:06',NULL,165,'2013/12/SRP001095','2010-02-26 00:00:00',NULL);
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (531,'UNIVERSITY OF WARSAW',NULL,1,NULL,'Mice of 2 genotypes followed 2 different diets.','ERP011824','different diet treatments','FINISHED','SUBMITTED','ilona.grabowicz@mimuw.edu.pl','Ilona Grabowicz','2016-01-20 14:12:06','Webin-41354',422,'2015/09/ERP011824','2015-08-26 00:00:00','PRJEB10562');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (532,'Aberystwyth University',NULL,1,'2015-10-27','It has been suggested that the rumen microbiome and rumen function might be disrupted if methane production in the rumen is decreased. Furthermore concerns have been voiced that geography and management might influence the underlying microbial population and hence the response of the rumen to mitigation strategies. Here we report the effect of the dietary additives: linseed oil and nitrate on methane emissions, rumen fermentation, and the rumen microbiome in two experiments from New Zealand (Dairy 1) and the UK (Dairy 2). Dairy 1 was a randomized block design with 18 multiparous lactating cows. Dairy 2 was a complete replicated 3 x 3 Latin Square using 6 rumen cannulated, lactating dairy cows. Treatments consisted of a control total mixed ration (TMR), supplementation with linseed oil (4% of feed DM) and supplementation with nitrate (2% of feed DM) in both experiments. Methane emissions were measured in open circuit respiration chambers and rumen samples were analyzed for rumen fermentation parameters and microbial population structure using qPCR and next generation sequencing (NGS). Supplementation with nitrate, but not linseed oil, decreased methane yield (g/kg DMI; P<0.02) and increased hydrogen (P<0.03) emissions in both experiments. Furthermore, the effect of nitrate on gaseous emissions was accompanied by an increased rumen acetate to propionate ratio and consistent changes in the rumen microbial populations including a decreased abundance of the main genus Prevotella and a decrease in archaeal mcrA (log10 copies/ g rumen DM content). These results demonstrate that methane emissions can be significantly decreased with nitrate supplementation with only minor, but consistent, effects on the rumen microbial population and its function, with no evidence that the response to dietary additives differed due to geography and different underlying microbial populations.','ERP011843','Does dietary mitigation of enteric methane production affect rumen function and animal productivity in dairy cows?','FINISHED','SUBMITTED','cjn@aber.ac.uk','Charles Newbold','2016-01-20 14:12:06','Webin-41399',434,'2015/09/ERP011843','2015-08-27 00:00:00','PRJEB10579');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (533,'WEIZMANN INSTITUE OF SCIENCE',NULL,1,NULL,'All domains of life feature diverse molecular clock machineries that synchronize physiological processes to diurnal environmental fluctuations. However, no mechanisms are known to cross-regulate prokaryotic and eukaryotic circadian rhythms in multi-kingdom ecosystems. Here, we show that the intestinal microbiota, in both mice and humans, exhibits diurnal oscillations that are influenced by feeding rhythms, leading to time-specific compositional and functional profiles over the course of a day. Ablation of host molecular clock components or induction of jet lag leads to aberrant microbiota diurnal fluctuations and dysbiosis, driven by impaired feeding rhythmicity. Consequently, jet lag-induced dysbiosis in both mice and humans promotes glucose intolerance and obesity that are transferrable to germ-free mice upon fecal transplantation. Together, these findings provide the first evidence of coordinated meta-organism diurnal rhythmicity, and offer a microbiome-dependent mechanism for common metabolic disturbances in humans with aberrant circadian rhythms, such as those documented in shift workers and frequent flyers.','ERP006800','Trans-kingdom control of microbiota diurnal oscillations promotes metabolic homeostasis','FINISHED','SUBMITTED','christoph.thaiss@weizmann.ac.il','Christoph Thaiss','2016-01-20 14:12:06','Webin-38594',428,'2015/09/ERP006800','2014-08-29 00:00:00','PRJEB7112');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (535,'Centro de Biotecnologia Alberto Ruiz',NULL,1,NULL,'Study on the microbial community related to de bioleaching processes of copper sulfide ore.','ERP010956','Study on the microbial community of a bioleaching column test','FINISHED','SUBMITTED','macostag@ucn.cl','Mauricio Acosta Grinok','2016-01-20 14:12:06','Webin-41043',26,'2015/09/ERP010956','2015-07-06 00:00:00','PRJEB9810');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (536,'UO',NULL,1,NULL,'Samples were collected from a series of hot springs at the headwaters of Little Hot Creek located in the Long Valley Caldera near Mammoth Lake, CA. The springs are supersaturated for carbonate, circumneutral in pH, and range from a temperature of 50? to 80? C. Genomic DNA was isolated from biofilms and water filtered through a PES 0.22?m 13mm filter using the Zymo Xpedition Soil/Fecal kit (Zymo Research Corp., Irvine, CA). Sequencing was conducted using either MiSeq PE250, PE300, or HiSeq Rapid PE250. MiSeq runs were prepared using the Agilent SureSelect kit (Agilent Technologies, Irvine, CA) with an approximate insert size of 400bp, and HiSeq PE250 samples were prepared using the Nextera XT library preparation kit (Illumina, San Diego, CA).','ERP011237','The 2014 and 2015 International Geobiology Courses took place at Little Hot Creek (LHC), California. The data shown here represents an attempt to gather information from a number of sources at LHC.','FINISHED','SUBMITTED','blake.w.stamps-1@ou.edu','Blake Stamps','2016-01-20 14:12:06','Webin-1078',197,'2015/09/ERP011237','2015-07-30 00:00:00','PRJEB10054');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (537,'PUBLIC HEALTH ENGLAND',NULL,1,NULL,'Availability of fast, high throughput and low cost whole genome sequencing (WGS) holds great promise within public health microbiology, with applications ranging from outbreak detection and tracking transmission events, to understanding the role played by microbial communities in health and disease. Within clinical metagenomics, identifying microorganisms from a complex and host enriched background remains a central computational challenge. In this study we sequenced two metagenomic samples, a known viral mixture of 25 human pathogens and an unknown complex biological mode using benchtop technology, highlighting the current challenges  and opportunities in this field.','ERP011248','In this study we describe some of the challenges of clinical metagenomics through benchtop sequencing of two distinct sample types, designed to highlight current application and opportunities in this field.','FINISHED','SUBMITTED','graham.rose@phe.gov.uk','Graham Rose','2016-01-20 14:12:06','Webin-38729',50,'2015/09/ERP011248','2015-07-31 00:00:00','PRJEB10064');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (538,'CSIRO ANIMAL FOOD & HEALTH SCIENCES',NULL,1,NULL,'Japanese goats fed a diet of 50% Timothy grass and 50% concentrate with increasing levels of the anti-methanogenic compound, bromochloromethane (BCM) were investigated with respect to the microbial population shifts in the rumen. Microbial ecology methods identified species that exhibited positive and negative responses to the increasing levels of BCM. The methane-inhibited rumen appeared to adapt to the higher H2 levels by shifting fermentation to propionate which was mediated by an increase in the population of hydrogen-consuming Prevotella and Selenomonas spp.','ERP011822','Analysis of the goat rumen microbial community following inhibition of methane formation by bromochloromethane (a halogenated methane analogue)','FINISHED','SUBMITTED','mar778@csiro.au','Gonzalo Martinez Fernandez','2016-01-20 14:12:06','Webin-41073',434,'2015/09/ERP011822','2015-08-26 00:00:00','PRJEB10560');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (539,'NANOBIOTEC',NULL,5,'2016-12-01','Hydrocarbon contamination is a threat resulting from human activity in Antarctica because of the low degradation rate due to the cold climate conditions and the seasonal freezing and thawing of soil in summer ice-free areas like the Antarctic Peninsula. Hydrocarbons can accidentally reach soil and sediments and distribute underground, likely affecting the biota and causing changes in bacterial communities. Monitoring and study of the distribution of hydrocarbons and the consequent response of the microbiota helps in the design of bioremediation strategies and elaboration of contingency plans, both required by the Antarcic Treaty.','ERP011407','Diesel contamination of Antarctic soils and sediments','FINISHED','SUBMITTED','svazquez@ffyb.uba.ar','Susana Vazquez','2016-01-20 14:12:06','Webin-41285',241,'2015/09/ERP011407','2015-08-05 00:00:00','PRJEB10201');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (540,'Institute of Environmental Science & Research',NULL,1,'2015-10-02','Argentine ants were collected from two nests located in Wellington, New Zealand. RNA was extracted from a pool of 30 ants collected from each of these nests. All RNA was then combined and subjected to de novo metagenomic analysis, using an Illumina MiSeq instrument. Multiple displacement amplification was used to increase the yield of transcribed DNA to meet the minimum input requirements of the Illumina MiSeq. The Illumina TruSeq DNA library preparation was used to prepare libraries, and produce 250 bp paired end reads.','ERP011266','The RNA metagenomes of Argentine ants (Linepithema humile) collected in New Zealand.','FINISHED','SUBMITTED','richard.hall@esr.cri.nz','Richard Hall','2016-01-20 14:12:06','Webin-41266',291,'2015/09/ERP011266','2015-08-03 00:00:00','PRJEB10079');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (542,'Swiss Federal Research Institute WSL',NULL,1,'2015-12-31','Microbial communities and soil carbon (C) have been shown to vary in response to increasing vegetation cover during soil development after deglaciation. However, little is known about the ability of microorganisms to utilize various C sources in glacier forefield soils. We supplied ecologically relevant 13C-labelled C sources (Chlorella, Penicillium and Festuca) to three distinct environments (supraglacial sediments, barren soils and vegetated soils) of the Damma glacier area to monitor 13CO2 production. We identified prokaryotic and fungal populations able to utilize these sources by using DNA-stable isotope probing coupled with Illumina MiSeq sequencing of ribosomal markers. A high initial 13CO2 pulse indicated that 13C-labelled microbial and plant material were consumed. The 13C-enriched DNA results indicated that betaproteobacterial taxa affiliated to the families Oxalobacteraceae and Comamonadaceae were important players in C utilization from different sources and present in all environments. In contrast, different fungal taxa played different roles in C degradation depending on the soil environment. Overall, our findings reveal that C utilization  is driven by similar prokaryotic populations along a glacier forefield, while the distribution of active fungal populations are more influenced by environmental factors.','ERP011856','Assimilation of C by microbial populations in glacial forefields','FINISHED','SUBMITTED','thomas.rime@wsl.ch','Rime Thomas','2016-01-20 14:12:06','Webin-38176',220,'2015/09/ERP011856','2015-08-31 00:00:00','PRJEB10588');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (543,'VIRGINIA TECH',NULL,1,'2016-01-30','this study contains 12 samples for the analysis of abundance of bacterial communities on human residues farm derived samples','ERP012163','Study of the abundance of bacteria from human samples','FINISHED','SUBMITTED','gustavo1@vt.edu','Gustavo  Arango','2016-01-31 00:05:03','Webin-41557',361,'2015/09/ERP012163','2015-09-15 00:00:00','PRJEB10865');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (544,'Aberystwyth University',NULL,1,'2015-11-01','It has been suggested that the rumen microbiome and rumen function might be disrupted if methane production in the rumen is decreased. Furthermore concerns have been voiced that geography and management might influence the underlying microbial population and hence the response of the rumen to mitigation strategies. Here we report the effect of the dietary additives: linseed oil and nitrate on methane emissions, rumen fermentation, and the rumen microbiome in two experiments from New Zealand (Dairy 1) and the UK (Dairy 2). Dairy 1 was a randomized block design with 18 multiparous lactating cows. Dairy 2 was a complete replicated 3 x 3 Latin Square using 6 rumen cannulated, lactating dairy cows. Treatments consisted of a control total mixed ration (TMR), supplementation with linseed oil (4% of feed DM) and supplementation with nitrate (2% of feed DM) in both experiments. Methane emissions were measured in open circuit respiration chambers and rumen samples were analyzed for rumen fermentation parameters and microbial population structure using qPCR and next generation sequencing (NGS). Supplementation with nitrate, but not linseed oil, decreased methane yield (g/kg DMI; P<0.02) and increased hydrogen (P<0.03) emissions in both experiments. Furthermore, the effect of nitrate on gaseous emissions was accompanied by an increased rumen acetate to propionate ratio and consistent changes in the rumen microbial populations including a decreased abundance of the main genus Prevotella and a decrease in archaeal mcrA (log10 copies/ g rumen DM content). These results demonstrate that methane emissions can be significantly decreased with nitrate supplementation with only minor, but consistent, effects on the rumen microbial population and its function, with no evidence that the response to dietary additives differed due to geography and different underlying microbial populations.','ERP011921','Does dietary mitigation of enteric methane production affect rumen function and animal productivity in dairy cows?','FINISHED','SUBMITTED','cjn@aber.ac.uk','Charles Newbold','2016-01-20 14:12:06','Webin-41399',434,'2015/10/ERP011921','2015-09-02 00:00:00','PRJEB10640');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (546,'BioProject',NULL,1,NULL,'Sequencing of DNA amplified from membrane vesicles isolated from the oceans','SRP031871','Metagenome sequencing of marine membrane vesicles','FINISHED','HARVESTED',NULL,NULL,'2016-01-20 14:12:06',NULL,132,'2015/10/SRP031871','2013-11-01 00:00:00','PRJNA222589');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (548,'European Commission, DG Joint Research Centre',NULL,1,'2016-12-05','Water samples have been collected at four sites in Croatia/Serbia onthe Danube river. Analysis of the samples\' 16S metagenomes should provide an indication how the bacterial community changes going from a relatively clean site to more pulluted sites close to industrial areas.','ERP012642','16S V3-V4 metagenome from four Danube water samples','FINISHED','SUBMITTED','armino@mclink.it','Armin Lahm','2016-02-03 10:02:08','Webin-37291',112,'2015/10/ERP012642','2015-10-06 00:00:00','PRJEB11278');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (551,'AAU, India',NULL,1,NULL,'Mocrobiome study of rumen','ERP012647','Rumen microbiome','FINISHED','SUBMITTED','24breddy@gmail.com','Bhaskar Reddy','2016-02-03 10:02:09','Webin-41071',434,'2015/10/ERP012647','2015-10-07 00:00:00','PRJEB11283');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (552,'AAU, India',NULL,1,NULL,'sequence submission to EBI metagenomics','ERP012815','Metagenomic workshop submission','FINISHED','SUBMITTED','24breddy@gmail.com','Bhaskar Reddy','2016-02-03 10:02:09','Webin-41071',434,'2015/10/ERP012815','2015-10-15 00:00:00','PRJEB11429');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (553,'rachelpjlai',NULL,1,NULL,'Total RNA from sputum collected from patients with active tuberculosis (TB) was analysed by RNA sequencing. The transcription phenotype of Mycobacterium tuberculosis (Mtb) was found to resemble that of actively replicating bacteria in laboratory culture. Transcription profiling of commensal bacteria in the sputum provided further provide details in of the sputum environment. Understanding of the microbiology and microbial ecology of Mtb after release from tuberculous lesions may reveal novel opportunities to modulate the efficiency of disease transmission.','ERP012221','Total RNA-Seq on sputum samples from patients with active tuberculosis','FINISHED','SUBMITTED','rachel.lai@crick.ac.uk','Rachel Lai','2016-02-03 10:02:08','Webin-41409',398,'2015/10/ERP012221','2015-09-21 00:00:00','PRJEB10919');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (554,'MAX PLANCK INSTITUTE FOR MARINE MICROBIOLOGY',NULL,1,'2016-12-11','Microbial processes are fundamental in nutrient cycling and remineralization in marine sediments. To understand how ocean acidification (OA) may influence sediment microbial diversity and activity, naturally CO2-rich sites are increasingly being used. However, the characterization of naturally CO2-rich sites is often limited to OA-related variables, neglecting additional variables that may obscure OA effects, especially when the CO2 increase is caused by geological processes. Here, we reevaluated the factors affecting changes in microbial communities at volcanic CO2 seeps in Papua New Guinea based on a comprehensive characterization of the conditions in the sediment. Microbial community composition was assessed using molecular fingerprinting and amplicon sequencing. pH was among the factors significantly, yet not mainly, explaining changes in microbial community composition. In-situ microprofiles and trace element concentrations further showed a variation in the strength of the hydrothermal signature of the sediment at similar pH allowing the identification of sites which may better represent future OA than others. At these sites, changes in the microbial community may have implications for element cycling in the sediment. We recommend focusing on a detailed environmental characterization in future OA research, to ensure better comparability between studies and a more reliable selection of naturally CO2-rich sites.','ERP012766','Microbial community composition in the sediment at naturally CO2-rich sites and its implications for ocean acidification research','FINISHED','SUBMITTED','chassenr@mpi-bremen.de','Christiane Hassenrueck','2016-02-03 10:02:08','Webin-38659',165,'2015/10/ERP012766','2015-10-12 00:00:00','PRJEB11384');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (558,'AAU, India',NULL,1,NULL,'Metagenomic study of rumen microbiome fed with different fiber proportion','ERP012964','Metagenomic study of rumen microbiome','FINISHED','SUBMITTED','24breddy@gmail.com','Bhaskar Reddy','2016-02-03 10:02:09','Webin-41071',434,'2015/11/ERP012964','2015-10-30 00:00:00','PRJEB11564');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (559,'GeoGenetics',NULL,1,NULL,'Metagenomics - Sierre Leone bat faecal metagenome','ERP013028','Metagenomics - bat faecal metagenome','FINISHED','SUBMITTED','kristinebohmann@gmail.com','Kristine Bohmann','2016-02-03 10:02:08','Webin-41934',428,'2015/11/ERP013028','2015-11-02 00:00:00','PRJEB11631');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (560,'UNIVERSITY OF WARWICK',NULL,1,NULL,'Sampling took place on 21 st November 2010. Soil cores were taken from agricultural (pasture) fields located within Northern Ireland amended with Nephrops carapaces. Carapaces were apllied to the field biannually from 2007 (~1.236 kg m-2). Total DNA was extracted using MPBio Fast DNA Kit for soil and sequenced using an Illumina MiSeq platform - paired end reads. The soil is slightly acidic.','ERP013031','Soil metagenome sampled from a chitin-amended agricultural field','FINISHED','SUBMITTED','i.d.e.lidbury@warwick.ac.uk','Ian Lidbury','2016-02-03 10:02:09','Webin-41851',239,'2015/11/ERP013031','2015-11-02 00:00:00','PRJEB11634');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (561,'INDIAN INSTITUTE OF CHEMICAL BIOLOGY',NULL,1,NULL,'lake water from eastern india is collected and metagenomic study to be done.','ERP013033','metagenome sample_Rabindra sarovar lake','FINISHED','SUBMITTED','j12mishra@gmail.com','Gyan Prakash Mishra','2016-02-03 10:02:09','Webin-39784',105,'2015/11/ERP013033','2015-11-03 00:00:00','PRJEB11636');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (562,'BioProject','High/low methane',1,NULL,'Rumen metagenome and metatranscriptome sequencing from low and high methane producing sheep.','SRP022254','Sheep rumen microbiome','FINISHED','HARVESTED',NULL,NULL,'2016-01-20 14:12:06',NULL,434,'2014/11/SRP022254','2013-05-25 00:00:00','PRJNA202380');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (563,'CCMAR',NULL,1,'2016-09-30','Marine sponges host abundant and diverse associated bacteria, some of which known to produce biologically active natural products. Although much research effort has been applied to cultivate sponge-associated microorganisms, we still lack the ability to capture the full phylogenetic breadth of this symbiotic consortium in the laboratory. This study uses a cultivation-independent, shotgun metagenomics approach to unveil the taxonomic and functional features of the model sponge host Spongia sp. and its environmental surroundings (seawater and sediments). Samples were taken off the coast of Algarve, South Portugal (North-east Atlantic), and subjected to Illumina sequencing after microbial metagenomic DNA extraction. Taxonomic and functional profiles were assessed using MG-RAST with the Genbank and COG databases, respectively. Most abundant phyla in the sponge microbiome were Proteobacteria, Actinobacteria, Firmicutes, Bacteroidetes, Chloroflexi, Cyanobacteria, Planctomycetes and Acidobacteria, in this order, altogether comprising 84.35% of all the classifiable sequence reads. In contrast, the dominant phyla in seawater were Bacteroidetes and Proteobacteria (89.3% of all classifiable reads), whereas sediments were dominated by Proteobacteria, Bacteroidetes, Planctomycetes, Actinobacteria and Firmicutes (85%). We found significantly higher frequencies of transcription and translation termination factors in sediments and seawater, respectively, than in marine sponge microbial metagenomes. Moreover, higher abundances of motility and chemotaxis gene reads were found in the former microhabitats in comparison with the marine sponge microbiome, where a conspicuously higher frequency of Clustered Regularly Interspaced Palindromic Repeats (CRISPRs), accompanied by much lower viral abundance, was otherwise observed. These results suggest a highly distinct microbiome inhabiting Spongia sp., both in terms of taxonomy and function, characterized by the prevalence of k-strategy prokaryotes which collectively possess a sophisticated, early-branched viral defence system.','ERP012972','Microbial metagenome sequencing from marine sponges (Spongia sp.), seawater and sediments','FINISHED','SUBMITTED','miguelramos22@gmail.com','Miguel Ramos','2016-04-01 00:05:05','Webin-41873',481,'2015/11/ERP012972','2015-10-30 00:00:00','PRJEB11585');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (564,'IMPERIAL COLLEGE LONDON',NULL,1,NULL,'NeoM','ERP012953','NeoM','FINISHED','SUBMITTED','k.sim@imperial.ac.uk','Kathleen Sim','2016-02-03 10:02:09','Webin-41943',361,'2015/11/ERP012953','2015-10-29 00:00:00','PRJEB11554');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1379,'BGI',NULL,1,NULL,'GD_MGS_describe','ERP020679','GD_MGS_title','FINISHED','SUBMITTED','sun010848@gmail.com','xq sun','2017-01-09 10:28:42','Webin-45726',82,'2017/01/ERP020679','2017-01-09 10:28:42','PRJEB18726');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1570,'EESC/USP',NULL,1,NULL,'Through the pyrosequencing analysis of samples from support mateial and phase separator from the anaerobic fluidized bed reactor total of 83 genera was found, which 18 are involved to the nonionic surfactant degradation as well as its byproducts. The genera with the highest relative abundance were Sporomusa, Geobacter, Desulfobulbus, Synergistes, Sedimentibacter, Holophaga, Serpens and Azonexus. All samples showed high phylogenetic diversity in which found genera are somehow related to the degradation of LAE and were favored by the configuration of the anaerobic fluidized bed reactor.','ERP004413','Through the pyrosequencing analysis of samples from support mateial and phase separator from the anaerobic fluidized bed reactor total of 83 genera was found, which 18 are involved to the nonionic surfactant degradation.','FINISHED','SUBMITTED','jukawanishi@gmail.com','Juliana Braga','2017-03-13 13:56:06','Webin-1428',29,'2017/03/ERP004413','2017-03-13 13:56:06','PRJEB5055');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1587,'IRIS',NULL,1,NULL,'A crude oil contamination at sea causes changes in marine bacterial abundance and composition. Microorganisms that are initially undetectable in a pristine environment become prevalent following an exposure to petroleum compounds. The overarching goal of the study were examine through laboratory experiments changes in abundance and composition of bacteria. Here we present laboratory data obtained with crude oil in the range 30ppb ? 2000ppb (nominal concentration) during an 11 days exposure. Changes in chemical composition coincided with major changes in total number of bacteria as well as in bacterial composition.','ERP009442','Changeability of bacterial community composition caused by seawater exposure to crude oil at low temperature','FINISHED','SUBMITTED','adriana.krolicka@iris.no','Adriana Krolicka','2017-03-16 09:04:50','Webin-1612',27,'2017/03/ERP009442','2017-03-16 09:04:50','PRJEB8365');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1643,'LABORATORIO NACIONAL DE CIENCIA E TECNOLOGIA DO BI',NULL,1,NULL,'Thermophilic two-phase anaerobic digestion using innovative fixed-bed reactor for enhanced organic matter removal and bioenergy recovery from sugarcane vinasse','ERP015373','Thermophilic two-phase anaerobic digestion using innovative fixed-bed reactor for enhanced organic matter removal and bioenergy recovery from sugarcane vinasse','FINISHED','SUBMITTED','gabriela.persinoti@bioetanol.org.br','Gabriela Persinoti','2017-03-24 10:35:42','Webin-39952',29,'2017/03/ERP015373','2017-03-24 10:35:42','PRJEB13796');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1806,'BioProject',NULL,1,NULL,'Recycled paper sludge microbial community as a potential source of enzymes','SRP095053','Paper sludge microbial community','FINISHED','HARVESTED',NULL,NULL,'2017-06-13 15:54:03','Webin-842',26,'2017/05/SRP095053','2017-06-13 15:54:03','PRJNA356078');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1820,'TECH_DENMARK',NULL,1,NULL,'The bacterial and archaeal communities in a bioelectrochemical-anaerobic diegstion-coupled system were sequenced by Ion Torrent Personal Genome Machine','DRP001287','An innovative bioelectrochemical-anaerobic digestion-coupled system for counteracting ammonia toxicity by in-situ ammonia recovery: process performance and microbial ecology','FINISHED','HARVESTED',NULL,NULL,'2017-06-20 13:53:13','Webin-842',26,'2017/05/DRP001287','2017-06-20 13:53:13','PRJDB2641');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1840,'BioProject',NULL,1,NULL,'Amplicon and shotgun sequencing of microbiome in two sugarcane filter cake compost piles were carried out to analyse the dynamics of fungal and bacterial communities along the process and biomass degrading profile for second generation bioethanol.','SRP059258','Sugarcane filter cake compost raw sequence reads','FINISHED','HARVESTED',NULL,NULL,'2017-06-20 14:34:13','Webin-842',29,'2017/05/SRP059258','2017-06-20 14:34:13','PRJNA285006');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (283,'MetaSoil',NULL,1,NULL,'The soil ecosystem is critical for human health, affecting aspects of the environment from key agricultural and edaphic parameters to critical influence on climate change. Soil has more unknown biodiversity than any other ecosystem. We have applied diverse DNA extraction methods coupled with high throughput pyrosequencing to explore 4.88109 bp of metagenomic sequence data from the longest continually studied soil environment (Park Grass experiment at Rothamsted Research in the UK). Results emphasize important DNA extraction biases and unexpectedly low seasonal and vertical soil metagenomic functional class variations. Clustering-based subsystems and carbohydrate metabolism had the largest quantity of annotated reads assigned although o50% of reads were assigned at an E value cutoff of 105. In addition, with the more detailed subsystems, cAMP signaling in bacteria (3.24±0.27% of the annotated reads) and the Ton and Tol transport systems (1.69±0.11%) were relatively highly represented. The most highly represented genome from the database was that for a Bradyrhizobium species. The metagenomic variance created by integrating natural and methodological fluctuations represents a global picture of the Rothamsted soil metagenome that can be used for specific questions and future inter-environmental metagenomic comparisons. However, only 1% of annotated sequences correspond to already sequenced genomes at 96% similarity and E values of o105, thus, considerable genomic reconstructions efforts still have to be performed. See http://www.genomenviron.org/Projects/METASOIL.html for additional details.','MetaSoil','MetaSoil','FINISHED','HARVESTED',NULL,NULL,'2016-01-20 14:12:06','Webin-21675',244,'2014/04/MetaSoil','2015-01-01 00:00:00',NULL);
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (334,'CCME-COLORADO','none',1,'2013-09-28','Although soil microbes play critical roles in terrestrial ecosystems, factors controlling spatial variability in the diversity and functional capabilities of these communities remain poorly understood. Particular knowledge gaps exist for biomes severely impacted by human activities, including the native tallgrass prairie that once covered >65 million ha of the midwestern United States, but has been nearly eradicated by decades of agricultural practices. Here we have reconstructed the microbial biodiversity that once sustained this highly productive ecosystem by analyzing soils from prairie relicts via shotgun metagenomics and targeted sequencing of the 16S rRNA gene. The taxonomic and functional diversity of the microbial communities were well-correlated, demonstrating that these communities are not functionally equivalent as has often been assumed - even small changes in belowground diversity can have important impacts on soil processes. As for many plant and animal communities, we could predict the structure and functional traits of the microbial communities from climatic conditions, allowing us to build predictive maps of soil biodiversity across the historical range of the tallgrass prairie ecosystem. The biogeographical patterns were largely driven by changes in the abundance of Verrucomicrobia, a poorly-studied bacterial phylum that dominated the prairie soils. The shotgun metagenomic data suggest that these spatial patterns were associated with shifts in soil carbon and nitrogen dynamics across the tallgrass prairie ecosystem. We show that we can use metagenomic data to reconstruct the belowground biogeochemical and diversity gradients that once existed, information that could be essential to guiding ongoing efforts to restore the threatened tallgrass prairie ecosystem.','ERP003954','Reconstructing the microbial diversity and function of pre-agricultural tallgrass prairie soils in the United States','FINISHED','HARVESTED',NULL,NULL,'2016-01-20 14:12:06',NULL,244,'2013/11/ERP003954','2013-09-28 00:00:00','PRJEB4646');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (339,'NIOO-KNAW','none',5,NULL,'dummy_study','ERP002026','Mattias de Hollander test submission','FINISHED','SUBMITTED','M.deHollander@nioo.knaw.nl','Mattias de Hollander','2016-05-16 00:05:04','Webin-532',244,'2014/02/ERP002026','2012-12-05 00:00:00','PRJEB3402');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (369,'CSIRO',NULL,1,'2016-03-14','The latter part of the Australian Millenium drought in 2007-2009 caused the acidification of acid sulfate soils in wetland and former floodplain soils, which pose threats to terrestrial and coastal ecosystems even after the recovery of surface flows and ground water levels. Drying and subsequent oxidation of ASS materials caused soil pH to drop to less than 4 (forming sulfuric materials) in some areas, triggering environmental problems such as land degradation, loss of native plants and animals, and release of heavy metals and metalloids into ground water, rivers and wetlands. To understand this microbially-mediated oxidation process, microbial communities were studied within an acidified acid sulfate soil profile, to identify key microorganisms involved in soil acidification. Six soil layers were sampled from a soil profile according to soil morphology at the most acidic locationin the field. Total DNA from soil samples was extracted using MO-BIO PowerMax? Soil DNA Isolation Kit and sequenced by Illumina Miseq (250PE) by The Ramaciotti Centre, NSW, Australia, prepared with a Nextera DNA Sample Preparation Kit. There were five steps of non-specific amplification involved in Nextera-Miseq sequencing for obtaining enough DNA for sequencing.','ERP005307','Microbial Community of Mobilong Acid Sulfate Soil depth profile using Metagenomics','FINISHED','SUBMITTED','xiaoyi.wang@csiro.au','Xiaoyi  Wang','2016-03-15 00:05:04','Webin-37215',244,'2014/04/ERP005307','2014-03-15 00:00:00','PRJEB5872');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (392,'CSIRO',NULL,1,'2014-07-26','A pre-investigation of the microbial composition of the acid sulfate soils were carried in Oct 2011 to obtain the microbial information before lime injection','ERP005908','Acid sulfate soil microbial profile - pre-investigation','FINISHED','SUBMITTED','xiaoyi.wang@csiro.au','Xiaoyi  Wang','2016-01-20 14:12:06','Webin-37215',244,'2014/07/ERP005908','2014-05-27 00:00:00','PRJEB6384');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (509,'UNIVERSITY COLLEGE LONDON',NULL,1,NULL,'Soil sample taken from wild flower meadow in Danesbury Park, Hertfordshire.','ERP011521','Soil sample taken from wild flower meadow for metagenome study.','FINISHED','SUBMITTED','natalie.dawson.09@ucl.ac.uk','Natalie Dawson','2016-01-20 14:12:06','Webin-33183',244,'2015/08/ERP011521','2015-08-12 00:00:00','PRJEB10291');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (692,'BioProject',NULL,1,NULL,'Central objective of this study is to compare different nucleic acid extraction methods with respect to abundance and diversity of 16S rRNA genes and transcripts.  Grassland soil derived from the German Biodiversity Exploratories Schw?bische Alb, Schorfheide Chorin and Hainich-D?n.','SRP015824','16S rRNA gene and transcript based analysis of different methods to extract DNA and RNA from soil','FINISHED','HARVESTED',NULL,NULL,'2016-04-15 08:57:02',NULL,244,'2016/04/SRP015824','2016-04-15 08:57:02','PRJNA175504');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (699,'BioProject',NULL,1,NULL,'In three California annual grassland soils, the present (DNA-based) and potentially active (RNA-based) soil bacterial and fungal communities were tracked over a summer field season and in response to a controlled rewetting of intact soil cores. Phylogenetic marker genes for bacterial (16S) and fungal (28S) RNA and DNA were sequenced and the abundance of these genes and transcripts were measured.','SRP016037','soil metagenomeTargeted Locus (Loci)','FINISHED','HARVESTED',NULL,NULL,'2016-04-15 09:23:53',NULL,244,'2016/04/SRP016037','2016-04-15 09:23:53','PRJNA176825');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (734,'BioProject',NULL,1,NULL,'Ammonia oxidizing archaeal communities from a semi-arid tall grass prairies in Oklahoma were sequenced to gauge the diversity and response to climate warming.','SRP018428','Kessler Farm Soil AOA communities Targeted Locus (Loci)','FINISHED','HARVESTED',NULL,NULL,'2016-04-19 11:18:16',NULL,244,'2016/04/SRP018428','2016-04-19 11:18:16','PRJNA188716');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (764,'JGI',NULL,1,NULL,'none provided','SRP008595','Wisconsin, Switchgrass soil metagenome reference core Project','FINISHED','HARVESTED',NULL,NULL,'2016-04-22 11:55:44',NULL,244,'2016/04/SRP008595','2016-04-22 11:55:44',NULL);
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (786,'BioProject',NULL,1,NULL,'Soils sequester and release substantial atmospheric carbon, but the biological responses of soils to rising CO2 are not well understood. Studying fungal community responses to past and future CO2 concentrations should provide insights into how soils affect the path of rising atmospheric CO2. We studied fungal communities in a grassland ecosystem exposed to a preindustrial-to-future CO2 gradient (250-500 ppm) on two soil types, a black clay and a sandy loam. The data uploaded are from 454 pyrosequencing of ITS-1 region rDNA, obtained from soil samples. The project is a collaboration between Duke University and the USDA-ARS Grassland, Soil and Water Research Laboratory in Temple, TX.','SRP026691','Prairie soil fungi, preindustrial-to-future CO2 gradient, Targeted Locus (Loci)','FINISHED','HARVESTED',NULL,NULL,'2016-05-02 12:27:16',NULL,244,'2016/05/SRP026691','2016-05-02 12:27:16','PRJNA205400');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (831,'BioProject',NULL,1,NULL,'Responses of soil-borne fungi to environmental manipulations (elevated temperature and increased intervals between precipitation events) were evaluated through ITS1 locus sequencing. <p> The project explores fungal community shifts in response to manipulation of environmental conditions on an ecosystem level. 23 soil samples were collected, total DNA isolated and ITS1 targets sequenced.; Project type: Targeted Locus (Loci); Target description: A total of 23 soil samples were collected in June 2006 from the established rainfall manipulation (RaMPs) experiment. Total environmental DNA was extracted and fungal ITS1 regions PCR-amplified with target specific primers. The amplicons were sequenced on GS20 platform.','SRP029919','RaMPs 2006 tallgrass prairie soil fungi ITS1 locus sequencing','FINISHED','HARVESTED',NULL,NULL,'2016-05-04 10:59:44',NULL,244,'2016/05/SRP029919','2016-05-04 10:59:44','PRJNA219046');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (888,'BioProject',NULL,1,NULL,'Different forms and rates of nitrogen addition indirectly effect the arbuscular mycorrhizal fungal community primarily by altering soil characteristics','SRP038013','Effects of N fertilization on arbuscular mycorrhizal fungal community in an alpine meadow on the Qinghai-Tibetan Plateau','FINISHED','HARVESTED',NULL,NULL,'2016-05-11 13:09:11',NULL,244,'2016/05/SRP038013','2016-05-11 13:09:11','PRJNA238405');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (906,'BioProject',NULL,1,NULL,'Microbial response to climate warming','SRP052767','Soil Random survey','FINISHED','HARVESTED',NULL,NULL,'2016-05-12 17:10:48',NULL,244,'2016/05/SRP052767','2016-05-12 17:10:48','PRJNA273431');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (944,'BioProject',NULL,1,NULL,'Evaluate the relative importance of soil characteristics, climate, and plant roots on rhizosphere community assembly.','SRP041697','Grassland 16S amplicons','FINISHED','HARVESTED',NULL,NULL,'2016-05-16 13:50:49',NULL,244,'2016/05/SRP041697','2016-05-16 13:50:49','PRJNA246258');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (952,'BioProject',NULL,1,NULL,'Further to a previous submission SRP030775, more community composition analyses of Stordalen Mire\'\'s thaw gradient.','SRP042265','Stordalen mire environmental genomics 2 Targeted Locus (Loci)','FINISHED','HARVESTED',NULL,NULL,'2016-05-16 15:21:26',NULL,244,'2016/05/SRP042265','2016-05-16 15:21:26','PRJNA235935');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1002,'BioProject',NULL,1,NULL,'Anthropogenic disturbance and global climate change are greatly affecting ecosystems worldwide by ways such as widespread nitrogen (N) deposition and changing precipitation regimes. It is predicted that the mean annual precipitation and N deposition in North China will continue to increase in future. Although much of work has been focus on the response of above-ground plant community to presumed climate change, the effects of nitrogen and water addition on the soil microbial community has not yet been systematically surveyed. Given the important role of soil microbial communities in nutrient cycling in terrestrial ecosystem, the effects of nitrogen and water addition and their interaction on soil microbial communities increased our attention. The specific aims of this project are: (1) to analyze the response of soil microbial diversity and community structure to the increase of soil N inputs and soil moisture; (2) to reveal the relationship between the variation of microbial communities and the above-ground plant community. The study sites were located in Duolun county, Inner Mongolia (E 116?17?20?, N 42?2?29?). A field experiment with a constant increase in precipitation and three Nitrogen supply levels was established in 2005 to examine the potential effects of N deposition and changes in precipitation regime on the ecological processes. A split-plot experimental design was employed in this study. The block was divided into two main plots with water treatment (ambient precipitation and water addition). Each main plot was divided into twenty-eight 8 m ? 8 m subplots with a 1-m wide buffer zone. Four levels of Nitrogen treatment (N0 (control), N5 (5 g N m-2 yr-1), N10 (10 g N m-2 yr-1), N15 (15 g N m-2 yr-1), each including seven replicates, were randomly assigned to each subplot within main plot. In the middle growing season from June to August, the water addition plots received 15 mm of precipitation weekly by sprinkling irrigation. A total of 180 mm precipitation, approximately 50% of mean annual rainfall, was added yearly during the growing season from 2005 to 2013.','SRP044829','Soil bacterial community in temperate grassland','FINISHED','HARVESTED',NULL,NULL,'2016-06-06 13:59:28',NULL,244,'2016/06/SRP044829','2016-06-06 13:59:28','PRJNA255542');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1003,'BioProject',NULL,1,NULL,'16s rRNA amplicon sequencing of DNA isolated from the Park Grass soil','SRP044877','16s rRNA profiling of Park Grass microbial community','FINISHED','HARVESTED',NULL,NULL,'2016-06-06 14:00:00',NULL,244,'2016/06/SRP044877','2016-06-06 14:00:00','PRJNA256215');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1019,'BioProject',NULL,1,NULL,'To contribute to the current understanding of free-living soil protozoan, we examined the diversity of two widespread and common groups of soil amoebae, the genus Acanthamoeba and the Myxomycetes, in the topsoil of grasslands from three German regions (Schorfheide-Chorin, Hainich-D?n, Schw?bische Alb). We choose these two taxa of Amoebozoa because of the converging evidence of their prevalence in soils and because they are overlooked by both traditional and molecular sampling methodologies: they are rarely, if ever, recovered using eukaryote-wide primers. We developed specific primers to target the variable region V2 in the first part of the small subunit of the ribosomal RNA gene, using Roche GS FLX high-throughput sequencing.','SRP046300','Metacommunity analysis of two protozoan taxa (Amoebozoa) in grassland soils','FINISHED','HARVESTED',NULL,NULL,'2016-06-06 14:06:24',NULL,244,'2016/06/SRP046300','2016-06-06 14:06:24','PRJNA259201');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (2011,'EMG',NULL,1,NULL,'The Mobilong Soil Profile Third Party Annotation (TPA) assembly was derived from the primary whole genome shotgun (WGS) data set PRJEB5872. This project includes samples from the following biomes : Grassland.','ERP104178','EMG produced TPA metagenomics assembly of the Microbial Community of Mobilong Acid Sulfate Soil depth profile using Metagenomics (Mobilong Soil Profile) data set','FINISHED','SUBMITTED',NULL,NULL,'2017-11-15 14:07:13','Webin-460',244,'2017/11/ERP104178','2017-11-15 14:07:13','PRJEB22497');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (993,'BioProject',NULL,1,NULL,'Soil samples have been collected in a truffle orchard located in the Vosges (France). DNA was extracted and used to decipher the structure of the bacterial communities along truffle maturation.','SRP044189','Rolainville truffle orchards Metagenome','FINISHED','HARVESTED',NULL,NULL,'2016-06-06 13:51:31',NULL,231,'2016/06/SRP044189','2016-06-06 13:51:31','PRJNA254788');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1005,'BioProject',NULL,1,NULL,'Biodiversity and climate change in european forests','SRP045166','Fungi Metagenome','FINISHED','HARVESTED',NULL,NULL,'2016-06-06 14:01:19',NULL,231,'2016/06/SRP045166','2016-06-06 14:01:19','PRJNA257238');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1013,'BioProject',NULL,1,NULL,'The flow of energy within an ecosystem can be considered either top-down, where predators influence consumers, or bottom-up, where producers influence consumers. Plethodon cinereus (Red-backed Salamander) is a terrestrial keystone predator who feeds on invertebrates within the ecosystem. We investigated the impact of the removal of P. cinereus on the detritivore food web in an upland deciduous forest in northwest Ohio, USA. A total of eight aluminum enclosures, each containing a single P. cinereus under a small log, were constructed in the deciduous forest. On Day 1 of the experiment, four salamanders were evicted from four of the eight enclosures. Organic matter and soil were collected from the center of each enclosure at Day 1 and Day 21. From each sample, DNA was extracted, fungal-specific amplification performed, and 454 pyrosequencing was used to sequence the nuclear ribosomal internal transcribed spacer (ITS2) region and partial ribosomal large subunit (LSU). Changes in overall fungal community composition or species diversity were not statistically significant between treatments. Statistically significant shifts in the most abundant taxonomic groups of fungi were documented in presence but not absence enclosures. We concluded that P. cinereus does not affect the overall composition or diversity of fungal communities, but does have an impact on specific groups of fungi. This study used a metagenomics-based approach to investigate a missing link among a keystone predator, P. cinereus, invertebrates, and fungal communities, all of which are critical in the detritivore food web.','SRP045746','Soil Samples Targeted Locus (Loci)','FINISHED','HARVESTED',NULL,NULL,'2016-06-06 14:04:08',NULL,231,'2016/06/SRP045746','2016-06-06 14:04:08','PRJNA259285');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1018,'BioProject',NULL,1,NULL,'Tracking the soil microflora components associated with fast- and slow-growing mycelium of summer truffle (Tuber aestivum)','SRP046077','Soil microflora Targeted Locus (Loci)','FINISHED','HARVESTED',NULL,NULL,'2016-06-06 14:06:02',NULL,231,'2016/06/SRP046077','2016-06-06 14:06:02','PRJNA259670');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1020,'BioProject',NULL,1,NULL,'The purpose of this study was to explore the diversity of ammonia-oxidizing microbes (bacteria and archaea) in the soils at Coweeta LTER, a research site in western North Carolina.','SRP046366','Forest Soil Targeted Locus (Loci)','FINISHED','HARVESTED',NULL,NULL,'2016-06-06 14:06:34',NULL,231,'2016/06/SRP046366','2016-06-06 14:06:34','PRJNA259883');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (941,'CUSB-HM',NULL,1,NULL,'Animal models are invaluable tools which allow us to investigate the microbiome-host dialogue. However, experimental design introduces biases in the data that we collect, potentially leading to false conclusions. With obesity at pandemic levels animal systems of this disease have been developed; we investigated the role of experimental design on one such rodent model.  We used 454 pyrosequencing to profile the faecal bacteria of obese (n = 6) and lean (homozygous n= 6; heterozygous n= 6) Zucker rats over a 10 week period, maintained in mixed-genotype cages, to further understand the relationships between the composition of the intestinal bacteria and age, obesity progression, genetic background and cage environment. Phylogenetic and taxon-based univariate and multivariate analyses (non-metric multidimensional scaling, principal component analysis) showed that age was the most significant source of variation in the composition of the faecal microbiota. Second to this, cage environment was found to clearly impact the composition of the faecal microbiota, with samples from animals from within the same cage showing high community structure concordance. Importantly, the obese phenotype was not found to impact the faecal bacterial profiles. These findings demonstrate that the age and local environmental cage variables were driving the composition of the faecal bacteria and were more deterministically important than host genotype. These findings have major implications for understanding the significance of functional metagenomic data in experimental studies and beg the question; what is being measured in animal experiments in which different strains are housed separately, nature or nurture?','ERP005417','We analysed the gut microbiota of rats kept in the same cage','FINISHED','HARVESTED',NULL,NULL,'2016-05-16 09:56:40',NULL,421,'2016/05/ERP005417','2016-05-16 09:56:40','PRJEB5969');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1079,'FRED HUTCHINSON CANCER RESEARCH CENTER',NULL,1,NULL,'The FDA approved drug rapamycin increases lifespan in rodents1 and improves measures of age-related function in rodents2 and humans3; thus, rapamycin has promising translational potential as an intervention to promote both longevity and healthspan. Nevertheless, significant questions remain regarding the optimal dose, treatment regimen, efficacy, and safety of rapamycin administration in the context of healthy ageing4. Here we show that 3 months of rapamycin treatment is sufficient to increase life expectancy by more than 50% and improve measures of healthspan in middle-aged mice. Transient treatment with rapamycin is also associated with a remodelling of the microbiome, including dramatically increased prevalence of segmented filamentous bacteria in the small intestine. We also define a dose in female mice that does not extend lifespan. This higher dose of rapamycin is associated with a striking shift in cancer prevalence toward aggressive haematopoietic cancers and away from non-haematopoietic malignancies in females. These data demonstrate that a single short-term rapamycin treatment late in life can be effective at delaying ageing and suggest that a similar regimen may be applicable to promote healthy ageing in people.','ERP014805','Transient rapamycin treatment robustly increases lifespan and healthspan in middle-aged mice','FINISHED','SUBMITTED','myajima@fredhutch.org','Masanao Yajima','2016-06-16 16:54:30','Webin-43431',421,'2016/04/ERP014805','2016-06-16 16:54:30','PRJEB13256');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1214,'PORTO CONTE RICERCHE SRL',NULL,1,NULL,'Comparison between shotgun metagenomic results obtained from paired mouse caecal contents and faeces','ERP017057','Comparison between mouse caecal contents and faeces','FINISHED','SUBMITTED','tanca@portocontericerche.it','Alessandro Tanca','2016-09-16 09:17:07','Webin-44750',421,'2016/09/ERP017057','2016-09-16 09:17:07','PRJEB15341');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1962,'KYUNG HEE UNIVERSITY',NULL,1,NULL,'Dietary fat-induced gut barrier dysfunction and pathobiont expansion aggravate experimental colitis','ERP103942','Dietary fat-induced gut barrier dysfunction and pathobiont expansion aggravate experimental colitis','FINISHED','SUBMITTED','cannavarosoo@gmail.com','Min-Soo Kim','2017-10-04 13:48:10','Webin-36326',421,'2017/10/ERP103942','2017-10-04 13:48:10','PRJEB22269');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (505,'HHMI-UNC',NULL,1,NULL,'Land plants associate with a root microbiota distinct from the complex microbial community present in surrounding soil. The microbiota  colonizing the rhizosphere (immediately surrounding the root), and the endophytic compartment (within the root), contribute to plant growth, productivity, carbon sequestration, and phytoremediation1,2,3. Colonization of the root occurs despite a sophisticated plant immune system4,5, suggesting finely-tuned discrimination of mutualists and commensals from pathogens. Genetic principles governing the derivation of host-specific endophyte communities from soil communities are poorly understood. We pyrosequenced the bacterial 16S rRNA gene of >600 Arabidopsis thaliana plants to test the hypotheses that the root rhizosphere and endophyte compartment microbiota of plants grown under controlled conditions in natural soils are (i) sufficiently dependent on the host to remain consistent across different soil types and developmental stages, and (ii) sufficiently dependent on host genotype to vary between inbred Arabidopsis accessions. We describe different bacterial communities in two geochemically distinct bulk soils, and in rhizosphere and endophyte compartments prepared from roots grown in these soils. The communities in each compartment are strongly influenced by soil type. Endophyte compartments from either soil feature overlapping low-complexity communities that are markedly enriched for Actinobacteria and specific families from other phyla, notably Proteobacteria. Some bacteria vary quantitatively between plants of different developmental stages and genotypes. Our work provides unprecedented rigor to define an endophyte compartment microbiome, facilitating controlled dissection of plant-microbe interactions derived from complex soil communities.','ERP001384','Defining the core Arabidopsis thaliana root microbiome','FINISHED','SUBMITTED','yourston@email.unc.edu','Scott  Yourstone','2016-01-20 14:12:06','Webin-231',466,'2015/07/ERP001384','2012-04-26 00:00:00','PRJEB2989');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (519,'UNIVERSITY OF VERONA',NULL,1,'2016-10-06','A whole metagenome sequencing approach was carried out, aiming at the taxonomic identification of the microbial components of berry ecosystem and to unveil whether the variations of environmental withering conditions could lead to significant modifications of microbial consortia and metabolic pathways on grape berries.','ERP011461','Study of microbial consortia of withered berries of cv. Corvina using whole metagenome sequencing approach','FINISHED','SUBMITTED','ilenia.campedelli@gmail.com','Ilenia Campedelli','2016-01-20 14:12:06','Webin-41272',466,'2015/09/ERP011461','2015-08-07 00:00:00','PRJEB10238');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (613,'BANGOR UNIVERSITY',NULL,1,NULL,'The aim is to comparatively analyze the microbiome found in oak trees using metagenomics. Field samples have been taken from healthy oak trees and trees suffering from the disease acute oak decline. Acute oak decline is a spreading disease in Europe and North America, causing wilting and potentially death of oaks.','ERP014260','Analysis of metagenomes from oak wood taken from healthy and diseased trees.','FINISHED','SUBMITTED','m.broberg@bangor.ac.uk','Martin Broberg','2016-02-22 11:13:49','Webin-42981',466,'2016/02/ERP014260','2016-02-22 11:13:49','PRJEB12757');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (614,'UNIVERSITY OF EDINBURGH',NULL,1,'2017-01-20','This project is a combined ecological and molecular investigation into the diversity of the Kuwaiti Flora and the soil seed bank. It is aimed at extracting DNA barcode regions from plant leaves to build a DNA reference library. Also, to investigate the diversity of species present in the soil by DNA extracting soil samples and using tools such as metagenomics and metabarcoding to verify the utility of the database.','ERP014120','DNA barcoding the flora of Kuwait','FINISHED','SUBMITTED','mabdullah@rbge.ac.uk','Mansour Abdullah','2016-02-26 14:57:55','Webin-42912',466,'2016/02/ERP014120','2016-02-26 14:57:55','PRJEB12627');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (832,'BioProject',NULL,1,NULL,'We aimed at analyzing the composition and structure of the phyllosphere bacterial communities of W. australiana and addressed two major questions: (1) what is the specialty of the bacterial community in the phyllosphere of floating macrophytes; (2) what controls the bacterial communities of the macrophyte phyllosphere in paddy soils.  16S rDNA V3 section of bacteria associated with phyllosphere of Wolffia australiana, water, and soil in paddy-soil ecosystem.','SRP029929','Phyllosphere bacterial community of Wolffia australiana Targeted Locus (Loci)','FINISHED','HARVESTED',NULL,NULL,'2016-05-04 11:05:26',NULL,466,'2016/05/SRP029929','2016-05-04 11:05:26','PRJNA219151');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1163,'UNIV-WUE',NULL,1,NULL,'Carnivorous plants of the genus Nepenthes have been studied for over a century, but surprisingly little is known about associations with microorganisms. The two species Nepenthes rafflesiana and Nepenthes hemsleyana differ in their primary nutrient source, sequestering nitrogen from arthropod prey and bat faeces, respectively. We expected bacterial communities living in the pitchers to resemble this diet difference. Samples were taken from different parts of the pitchers (leaf, peristome, inside, outside, digestive fluid) of both species. Bacterial communities were determined using culture-independent high-throughput amplicon sequencing. Bacterial diversity and community structure was comparable across plant species and most tissues, except digestive fluids. These showed opposing trends with N. hemsleyana harbouring a more diverse bacterial community. In N. rafflesiana fluid, high levels of Acidocella spp. implied a close association with the plant. In N. hemsleyana fluid, vertebrate gut symbionts as well as saprophytic taxa could be detected, the latter of which might act as competitors for nutrients. Generally, digestive fluid communities were highly variable in structure, which might be applicable to a difference in digestion status. Nitrogen-fixing bacteria were present in both study species, which might provide essential nutrients to the plant at times of low prey capture and rare encounters with bats, respectively.','ERP008969','Diet determines bacterial diversity and community structure in Bornean pitcher plants','FINISHED','HARVESTED',NULL,NULL,'2016-06-29 16:39:01',NULL,466,'2016/06/ERP008969','2016-06-29 16:39:01','PRJEB7957');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1199,'FONDAZIONE EDMUND MACH - COMPUTATIONAL BIOLOGY DEP',NULL,1,NULL,'The fungal populations present in Vitis vinifera L. cv. Corvina grapes and musts were compared in two different years vintages. The comparison was carried out through ITS1-5.8S-ITS2 454-pyrosequencing. Grapes were collected from a recent vineyard located in the Italian winery region called \"Valpolicella\" (Sant\"Ambrogio, Verona, Italy). After the harvesting, grapes were subjected to withering in a dedicated warehouse located a few kilometers far from the vineyars (<5Km). The warehouse is supplemented with automatic systems able to control and modify the internal temperature and humidity. The duration of grape withering was defined according to the regional rules for Amarone production (Paronetto and Dellaglio, 2011), which define the time in which the grapes for Amarone vinification can be mashed. The comparison of the fungal populations present in these samples sheds lights on the Amarone production, allowing the identification of the persistent fungal genera and on genera varying as suffering from seasonal environmental changes.','ERP016938','Amplicon-based metagenomics analysis of Vitis vinifera L. cv. Corvina grapes and fresh musts','FINISHED','SUBMITTED','stefanini.irene@gmail.com','Irene Stefanini','2016-09-08 15:32:18','Webin-44201',466,'2016/08/ERP016938','2016-09-08 15:32:18','PRJEB15229');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1271,'FONDAZIONE EDMUND MACH - COMPUTATIONAL BIOLOGY DEP',NULL,1,NULL,'The fungal populations present in Vitis vinifera L. cv. Corvina grapes and musts were compared in two different years vintages. The comparison was carried out through ITS1-5.8S-ITS2 454-pyrosequencing. Grapes were collected from a recent vineyard located in the Italian winery region called \"Valpolicella\" (Sant\"Ambrogio, Verona, Italy). After the harvesting, grapes were subjected to withering in a dedicated warehouse located a few kilometers far from the vineyars (<5Km). The warehouse is supplemented with automatic systems able to control and modify the internal temperature and humidity. The duration of grape withering was defined according to the regional rules for Amarone production (Paronetto and Dellaglio, 2011), which define the time in which the grapes for Amarone vinification can be mashed. The comparison of the fungal populations present in these samples sheds lights on the Amarone production, allowing the identification of the persistent fungal genera and on genera varying as suffering from seasonal environmental changes.','ERP016297','metataxonomics analysis of fungal populations in Vitis vinifera L. cv. Corvina grapes and musts','FINISHED','SUBMITTED','stefanini.irene@gmail.com','Irene Stefanini','2016-10-18 14:14:51','Webin-44201',466,'2016/09/ERP016297','2016-10-18 14:14:51','PRJEB14642');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1323,'CEBITEC',NULL,1,NULL,'Taxonomic analysis of the microbial community in stored sugar beets using high-throughput sequencing of different marker genes','ERP013558','Taxonomic analysis of the microbial community in stored sugar beets using high-throughput sequencing of different marker genes','FINISHED','SUBMITTED','dwibberg@cebitec.uni-bielefeld.de','Daniel Wibberg','2016-11-28 10:42:04','Webin-754',466,'2016/11/ERP013558','2016-11-28 10:42:04','PRJEB12118');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1324,'FOOD AND ENVIRONMENT RESEARCH AGENCY',NULL,1,NULL,'Metabarcoding was used to study bacteria in oak samples. Samples were taken from oaks showing signs of acute oak decline. A general 16S rRNA gene primer was used followed by pyrosequencing.','ERP013941','Identification of the oak microbiome','FINISHED','SUBMITTED','msapp@web.de','Melanie Sapp','2016-11-28 10:42:44','Webin-40965',466,'2016/11/ERP013941','2016-11-28 10:42:44','PRJEB12457');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1335,'107622013210523',NULL,1,NULL,'This study addressed two fundamental questions: (1)How does the bacterial community structure differ between the different organs of Lepidium perfoliatum L.?(2) Do different tissues have thepotential to influence the structure of bacterial communities?','ERP013109','A diverse array of bacteria that inhabit the rhizosphere and different plant organs play a crucial role in plant health and growth.','FINISHED','SUBMITTED','820850287@qq.com','Cong Cheng','2016-12-02 10:35:32','Webin-42004',466,'2016/12/ERP013109','2016-12-02 10:35:32','PRJEB11701');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1337,'UNIV-WUE',NULL,1,NULL,'The crown gall disease in grapevine is mediated by Agrobacterium vitis and results in tumorous growth. The tumor offers an ecological niche for the pathogen and eventually for other bacteria as well. We wanted to know if the crown gall disease affects the bacterial community of the grapevine plant. Using next generation sequencing of the V4 region of 16S rDNA amplicons we recorded the microbiota from galled and non-galled grapevine plants as well as of the soil at three seasons of the year. The root, trunk, one-year-old cane and the soil hosted distinct microbiota, which differed according to spring, summer, and autumn. However, the crown gall disease influenced the structure of the microbiota only in grapevine trunks. There the bacterial richness was higher and comprised for example exclusively the plant pathogens Xanthomonas sp., Sodalis sp., and an unknown Gammaproteobacterium. The seasons additionally affected the microbiota in the trunks with and without a crown gall differently. A. vitis, Pseudomonas sp. and Enterobacteriaceae sp. were the most abundant sequences in crown galls at each season. In contrast, in healthy trunks (=non-galled) the microbial composition changed from season to season. Altogether, twenty-two operational taxonomic units, representing bacterial species, were significantly enriched in crown galls and only three in healthy trunks. Our results support the idea that other bacteria than the pathogen A. vitis can adapt to the crown gall environment. How these profit from the new environment and if they influence crown gall development will be the subject of future work.','ERP013477','Effects of crown gall disease on natural microbiota of Vitis vinifera','FINISHED','SUBMITTED','a.keller@biozentrum.uni-wuerzburg.de','Alexander Keller','2016-12-05 16:33:20','Webin-897',466,'2016/12/ERP013477','2016-12-05 16:33:20','PRJEB12040');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1347,'UNIVERSITY OF NORTH CAROLINA AT CHAPEL HILL',NULL,1,NULL,'Bacteria living on and in leaves and roots influence many aspects of their hosts? health, so the extent of plant genetic control over their microbiota is of great interest to crop breeders and evolutionary biologists. Laboratory-based studies may overestimate the influence of certain plant genes on the microbiome and fail to detect others, because they do a poor job simulating true environmental heterogeneity. We conducted a large-scale field experiment to disentangle the effects of genotype, environment, age, and year of harvest on bacterial communities associated with leaves and roots of Boechera stricta, a wild perennial mustard.  Genetic control of the microbiome varied substantially among sites, and bacterial community composition shifted as plants aged. Furthermore, a large proportion of leaf bacteria were shared with roots, suggesting inoculation from soil. Our results highlight the complexity of microbiome formation in natural environments.','ERP011833','16S rRNA amplicons (V4 region) of bacteria living on and in roots and leaves of Boechera stricta from field experiments in the Rocky Mountains','FINISHED','SUBMITTED','derek.lundberg@gmail.com','Derek Lundberg','2016-12-13 14:01:09','Webin-41440',466,'2016/12/ERP011833','2016-12-13 14:01:09','PRJEB10570');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1348,'MPIPZ',NULL,1,NULL,'Roots and leaves of healthy plants host taxonomically structured bacterial assemblies, and members of these communities contribute to plant growth and health. We established Arabidopsis leaf- and root-derived microbiota culture collections representing the majority of bacterial species that are reproducibly detectable by culture-independent community sequencing. We found an extensive taxonomic overlap between the leaf and root microbiotas. Genome drafts of 400 isolates revealed a large overlap of genome-encoded functional capabilities between leaf and root-derived bacteria with few significant differences at the level of individual functional categories. Using defined bacterial communities and a gnotobiotic Arabidopsis plant system we show that the isolates form assemblies resembling natural microbiotas on their cognate host organs, but are also capable of ectopic leaf or root colonization. Whilst this raises the possibility of reciprocal relocation between root and leaf microbiota members, genome information and recolonization experiments also provide evidence for microbiota specialization to their respective niche.','ERP012970','Arabidopsis thaliana culture collections of leaf- and root-associated bacteria  (Synthetic Communities 16S rRNA data)','FINISHED','SUBMITTED','garridoo@mpipz.mpg.de','Ruben Garrido-Oter','2016-12-13 14:11:25','Webin-41930',466,'2016/12/ERP012970','2016-12-13 14:11:25','PRJEB11583');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1358,'CHINESE ACADEMY OF SCIENCES',NULL,1,NULL,'The microorganisms inhabiting plant, such as leaf and root play important roles in plant growth. However, the microbial community structure and their response to climate changes are poorly understood. The objective was to survey the composition of microbial communities from root and clarify the response of bacterial communities to elevated CO2 and/or soil temperature.','ERP008578','Response of plant-associated bacterial community to simulated climate changes','FINISHED','SUBMITTED','bocai775@163.com','Ruibo Sun','2017-01-05 10:18:47','Webin-38711',466,'2017/01/ERP008578','2017-01-05 10:18:47','PRJEB7648');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1365,'TUGRAZ',NULL,1,NULL,'Africa is among the continents where the battle over genetically modified crops is currently being played out. The impact of GM in Africa could potentially be very positive. In Uganda, researchers have developed transgenic banana lines resistant to banana Xanthomonas wilt. The transgenic lines expressing hrap and pflp can provide a timely solution to the pandemic. However, the impact of the transgenes expression on non-target microorganisms has not yet been investigated. To study this effect, transgenic and control lines were grown under field conditions and their associated microbiome was investigated by 16S rRNA gene profiling combining amplicon sequencing and molecular fingerprinting. Three years after sucker planting, no statistically significant differences between transgenic lines and their non-modified predecessors were detected for their associated bacterial communities. The overall gammaproteobacterial rhizosphere microbiome was highly dominated by Xanthomonadales, while Pseudomonadales and Enterobacteriales were accumulated in the pseudostem. Shannon indices revealed much higher diversity in the rhizosphere than in the pseudostem endosphere. However, the expression of the transgenes did not result in changes in the diversity of Gammaproteobacteria, the closest relatives of the target pathogen. In this field experiment, the expression of the resistance genes appears to have no consequences for non-target rhizobacteria and endophytes.','ERP010526','Transgenic banana plants expressing Xanthomonas wilt resistance genes revealed a stable non-target bacterial colonization structure','FINISHED','SUBMITTED','martina.koeberl@tugraz.at','Martina Koeberl','2017-01-05 10:29:05','Webin-1632',466,'2017/01/ERP010526','2017-01-05 10:29:05','PRJEB9422');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1370,'UNIVERSITY OF BRITISH COLUMBIA',NULL,1,NULL,'Kelp forest ecosystems are biodiversity hotspots, providing habitat for dense assemblages of marine organisms and an important source of energy and nutrients for both marine and terrestrial food webs. The surfaces of kelps support diverse microbial communities that are essential for the growth and development of their hosts, and for facilitating the transfer of carbon from algal primary production to higher trophic levels. We quantified the diversity and distribution of bacterial communities present on the surfaces of eight sympatric species of kelp from four sites on the central coast of British Columbia. Patterns of host-asscoiated microbial diversity were compared with samples collected from seawater and rocky substrate. This study is among the first to quantify the distribution of microbial diversity across sympatric species within kelp forests, providing an additional level of data towards a more comprehensive understanding of these ecologically important habitats.','ERP016802','Kelp forest microbial diversity','FINISHED','SUBMITTED','morien@zoology.ubc.ca','Evan Morien','2017-01-05 10:40:04','Webin-43333',466,'2017/01/ERP016802','2017-01-05 10:40:04','PRJEB15100');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1372,'University of Jyvaskyla',NULL,1,NULL,'Evidence for the pivotal role of plant-associated bacteria to plant health and productivity has accumulated rapidly in the last years. However, several key questions remain unanswered, among which is the impact of climate zones on the plant-associated microbiota. Importantly, plant-bacterial interactions in wild plants and in arcto-alpine biomes have not been well studied as compared to those in tropical and temperate biomes. We hypothesized that the bacterial communities associated with pioneer plants in the former regions have major roles in plant health support, and this would be reflected in the formation of climate and host plant specific endophytic communities. We also hypothesized that these endophytes would have a tight association with their respective host plants. To examine the drivers of the plant-associated bacterial communities, we compared the latter in the perennial plants Oxyria digyna and Saxifraga oppositifolia - naturally growing in three arcto-alpine regions [Mayrhofen, Austria - alpine, Kilpisj?rvi, Finland - low Arctic and Ny-?lesund, Svalbard -highArctic] - with those in the corresponding bulk soils. As expected, the bulk soil bacterial communities in the three regions were significantly different. The relative abundances of Proteobacteria decreased progressively from the alpine to the high-arctic soils, whereas those of Actinobacteria increased. The candidate division AD3 and Acidobacteria abounded in the low Arctic. With respect to the endophytic bacterial communities, plant species and geographic region were the major determinants of the community structures. The plants in the alpine region had higher relative abundances of Proteobacteria, as compared to the plants from the low- and high-arctic regions which were dominated by Firmicutes. A highly conserved shared set of omnipresent bacterial taxa was found to occur in the two plant species, here denoted as the ?tight? core bacteriome. Burkholderiales, Actinomycetales and Rhizobiales were common members of this core, and these taxa were the main contributors to the differences in the endophytic bacterial community structures across compartments as well as regions. We postulate that this tight core is due to selection by the two plants, concomitant with a highly efficient adaptation of such ?core bacteriome? to the plant-associated habitats.','ERP019565','Plants Assemble Species Specific Bacterial Communities From Common Core Taxa in Three Arcto-Alpine Climate Zones','FINISHED','SUBMITTED','mano.joy@gmail.com','Manoj Kumar','2017-01-05 10:42:45','Webin-45234',466,'2017/01/ERP019565','2017-01-05 10:42:45','PRJEB17695');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1395,'Graz University of Technology',NULL,1,NULL,'The indigenous phyllosphere microbiome was identified as a key component for plant growth and health, and for positive effects on microbial diversity within a built environment. Nevertheless, there is still limited understanding of the phyllosphere microbiota and its driving factors. To study the variability of the microbiome in relation to plant genotype and climate, we investigated 14 phylogenetically diverse plant species grown under different controlled conditions in the greenhouses of the Botanical Garden in Graz (Austria). All investigated plants showed highly specific bacterial abundances of up to 106 CFU cm-2 on their leaves. Bacterial diversity (Shannon index H?: 4.1 ? 6.8) and number of putative OTUs (Chao 1: 501 ? 1,097) were strongly plant species-dependent, but comprised similar dominant phyla: Firmicutes, Proteobacteria and Actinobacteria. Non-metric multidimensional scaling and BIO-ENV analysis showed a higher correlation of community composition to plant genotype rather than the ambient climatic variables. The antagonistic potential of the phyllosphere microbiome towards the plant pathogen Botrytis cinerea measured by production of antifungal volatile organic compounds (VOCs) differed in a range of 2 up to 58% of the isolates. Frequently isolated VOCs produces were represented by Bacillus ssp., Stenotrophomonas rhizophila and Kocuria ssp. Surprisingly, the applied biopesticide B. thuringiensis Bt407 was found well-established in all phyllospheres. This study indicates that indoor ornamentals feature a distinct, stable microbiota with a high proportion of antifungal VOC-producers on leaves irrespective of the indoor climate. Hence, the microbiome of phyllospheres stabilize microbial diversity indoors, and can lead to beneficial effects on human health inside buildings.','ERP014856','The plant is crucial: specific composition and function of the phyllosphere microbiome of indoor ornamentals','FINISHED','SUBMITTED','rocelamorortega@yahoo.com','Rocel Amor Ortega','2017-01-11 09:26:59','Webin-43485',466,'2017/01/ERP014856','2017-01-11 09:26:59','PRJEB13300');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1412,'STATION BIOLOGIQUE DE ROSCOFF',NULL,1,NULL,'We examined the bacterial flora associated with 20 cultured strains of Ectocarpus sp. from different locations, and recorded both the bacterial diversity associated directly to the alga and that found in the culture medium. Experiments were carried out to amplify an approximately 400 bp fragment of the bacterial 16S comprising the V3 and V4 regions and sequenced on the Illumina MiSeq platform using V2 reagents ( 2x250 bp reads).','ERP004961','16S metabarcoding of bacteria associated with cultured strains of the brown alga Ectocarpus sp.','FINISHED','SUBMITTED','simon.dittami@gmail.com','Simon Dittami','2017-01-12 10:43:38','Webin-32203',466,'2017/01/ERP004961','2017-01-12 10:43:38','PRJEB5542');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1552,'IUBT',NULL,1,NULL,'Bryophytes belong to the phylogenetically oldest land plants. The bryophyte genus Sphagnum is the dominant component of the mire and bog vegetation, which are of high importance for our world climate. Recently, it was shown that Sphagnum mosses are colonized in high abundances with special microbial communities, which fulfil important functions for the moss plantlets as well as for the whole bog ecosystem. However, data obtained resulted in a lot of new scientific questions; which are the objective of the current project. The major aim of this project was to illuminate the biodiversity and ecological characteristics of the Sphagnum-associated microbiome as a bio-resource.','ERP003972','Microbial diversity associated with Sphagnum mosses','FINISHED','SUBMITTED','anastasia.v.bragina@gmail.com','Anastasia Bragina','2017-03-07 11:42:24','Webin-1221',466,'2017/03/ERP003972','2017-03-07 11:42:24','PRJEB4660');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1568,'MQ',NULL,1,NULL,'Large-scale sequencing of nucleic acids extracted from diverse sources is now widely used to study the structure and function of bacterial communities. However, the biological complexity of natural environments, combined with both technical and practical difficulties in analysing large datasets often means the full potential of this technique is rarely achieved. In this respect, we present a targeted metagenomic pipeline, focusing on extracting one component of a complex bacterial community. We used a 3 step process: flow cytometric sorting, followed by direct whole genome amplification on purified cells, then high throughput sequencing. We selected Synechococcus, a ubiquitous marine picophytoplankton, as a target and tested our methods on defined mixtures of cultivated isolates before applying the approach on environmental samples. We show that this targeted approach can provide precise and in-depth assessment of the genomic potential of a selected group where low environmental abundance would limit the return of metagenomic information from community-wide DNA surveys. This approach will help to explore fine-scale population structure and biogeography in a genomic context and therefore the underlying mechanisms that allow specific keystone populations, e.g., marine Synechococcus, to cohabit and contribute to nutrient cycling across a range of habitats.','ERP004475','Targeted population genomics of marine Synechococcus cyanobacteria','FINISHED','SUBMITTED','martin.ostrowski@mq.edu.au','Martin Ostrowski','2017-03-13 13:51:12','Webin-1571',466,'2017/03/ERP004475','2017-03-13 13:51:12','PRJEB5110');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1625,'UNIVERSITY OF WESTERN ONTARIO',NULL,1,'2017-06-01','Common buckthorn (Rhamnus cathartica) is a competitive Eurasian woody shrub currently invading North America. Buckthorn thickets reduce native diversity and may reduce mycorrhizal diversity through the release of allelochemicals. Two aspects of buckthorn\'s invasional biology are explored: 1) determining buckthorn\'s allelochemical impacts on arbuscular mycorrhizae in mature sugar maple (Acer saccharum) stands, and 2) assessing possible changes in mycorrhizal communities in sugar maple soils in an open-greenhouse experiment. Data from invaded and uninvaded sugar maple soils revealed that arbuscular mycorrhizal fungi (AMF) diversity fluctuated as a function of season, sampling site or potting disturbance, but the presence of buckthorn had little effect on AMF development in maple roots. Buckthorn may be a mycorrhizal generalist, and changes in AMF abundance may be more influenced by underlying stochastic soil processes and aboveground plant composition than by buckthorn and its allelochemicals.','ERP021783','Fungi associated with Rhamnus cathartica in Southwestern Ontario','FINISHED','SUBMITTED','nweerasu@uwo.ca','Nimalka Weerasuriya','2017-03-22 10:18:29','Webin-39347',466,'2017/03/ERP021783','2017-03-22 10:18:29','PRJEB19724');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1645,'UCSDMI',NULL,1,NULL,'Microbial activity is an inherent feature of wine production. The microbial communities of grapes present regionally defined patterns, influenced by vineyard and climatic conditions. However, the degree to which these microbial patterns associate with the chemico-sensory qualities of wine is unclear. We demonstrate that both grape microbiota and wine metabolite profiles distinguish growing sub-regions and individual vineyards within Napa and Sonoma County, California. The bacterial and fungal consortia of wine fermentations, composed from vineyard and winery sources, correlate closely with the chemical composition of the finished wines. Grape must microbiota serve as highly accurate biomarkers for predicting metabolite abundance in finished wines using machine-learning models. The use of pre-harvest microbiota as an early predictor of wine qualities is unprecedented and poses a new paradigm for quality control of agricultural products. These findings add further evidence that microbial activity is a definable feature that quantitatively contributes to wine terroir.','ERP015814','Microbial biogeography of grapes predicts regional metabolite patterns in wine','FINISHED','SUBMITTED','glackermann@ucsd.edu','Gail Ackermann','2017-03-24 10:47:09','Webin-34',466,'2017/03/ERP015814','2017-03-24 10:47:09','PRJEB14186');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1676,'BioProject',NULL,1,NULL,'Baseline survey of the anatomical microbial ecology of an important food plant: Solanum lycopersicum (tomato).  Targeted and whole metagenomes from different organs of the tomato plant.','SRP017205','Tomato phyllosphere Metagenome','FINISHED','HARVESTED',NULL,NULL,'2017-03-30 15:55:03','Webin-842',466,'2017/03/SRP017205','2017-03-30 15:55:03','PRJNA179134');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1724,'CORVINUS UNIVERSITY OF BUDAPEST',NULL,1,NULL,'Seven Hungarian wine regions have been sampled for leaf and 20cm deep soil samples from the same plants. All vineyards are of different soil types and grape cultivars. Total DNA has been extracted and 16S amplicon sequencing has been applied to the samples.','ERP010802','16S amplicon based soil and leaf microbiome survey in Hungarian vineyards','FINISHED','SUBMITTED','tamas.deak@uni-corvinus.hu','Tamas Deak','2017-05-02 11:30:50','Webin-36792',466,'2017/05/ERP010802','2017-05-02 11:30:50','PRJEB9671');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1729,'UNIV-WUE',NULL,1,NULL,'Molecular sequencing techniques help to understand microbial biodiversity with regard to species richness, assembly structure and function. In this context, available methods are barcoding, metabarcoding, genomics and metagenomics. The first two are restricted to taxonomic assignments, while genomics only refers to functional capabilities of a single organism. Metagenomics by contrast yields information about organismal and functional diversity of a community. However currently it is very demanding regarding labour and costs and thus not applicable to most laboratories. Here, we show in a proof-of-concept that computational approaches are able to retain functional information about microbial communities assessed through 16S rDNA (meta)barcoding by referring to reference genomes. We developed an automatic pipeline to show that such integration may infer preliminary or supplementary genomic content of a community. We applied it to two biological data sets and delineate significantly overrepresented protein families between communities. The script alongside supporting data is available at http://bioapps.biozentrum.uni-wuerzburg.de.','ERP006146','Computational integration of genomic traits into 16S rDNA microbiota sequencing studies','FINISHED','SUBMITTED','a.keller@biozentrum.uni-wuerzburg.de','Alexander Keller','2017-05-04 09:22:11','Webin-897',466,'2017/05/ERP006146','2017-05-04 09:22:11','PRJEB6599');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1794,'BioProject',NULL,1,NULL,'This study is about testing an artificially elevated JA signalling on the wheat associated microbial community structure','SRP100290','wheat associated microbiome Raw sequence reads','FINISHED','HARVESTED',NULL,NULL,'2017-06-09 14:45:12','Webin-842',466,'2017/05/SRP100290','2017-06-09 14:45:12','PRJNA351276');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (293,'UH/IB','marination',1,'2012-05-01','We investigated the effect of marination on microbiota in modified athmosphere packaged broiler meat using metagenomics. To be able to see the differences in lactic acid bacteria communities on a strain level and to evaluate congruence between the methods, we characterized the lactic acid bacteria communities also by identification of isolates. Most raw poultry sold in Finland at the retail level is mixed with marinades containing oil, sugar, spices and acetic acid and packaged under modified atmosphere. Growth of Leuconostoc spp. has been observed to cause premature spoilage  of marinated broiler meat packages. In this study we investigated whether marinating enriched Leuconostoc spp. in meat microbiota. To obtain a complete view of the microbiota, we sequenced total DNA and 16S rRNA gene amplicons from the microbial communities. In addition, the lactic acid bacterial communities were characterized by identification of colonies. The results showed that marinade increased the proportions of the spoilage-associated Leuconostoc gasicomitatum in the communities as well as the proportions of L. gelidum and Lactobacillus spp.. The proportions of Carnobacterium, Vagococcus, Brochothrix thrermosphacta, Clostridium, Enterobacteriaceae and Vibrio were diminished. Analysis of 16S rRNA gene amplicons resulted in 312 and 284 operational taxonomical units (dissimilarity 0.03) in unmarinated and marinated meat, respectively, indicating that the meat communities were more diverse than hitherto shown. A number of bacterial taxa that have not been associated with late shelf-life meat before were detected, including Vagococcus and Vibrio that belonged to the predominating part of the microbial community in unmarinated meat. According to the functional analysis of the metagenomes, both communities were characterized by high proportions (15.6% or 17.9%) of genes involved in carbohydrate metabolism.','ERP001021','Comparison of microbial communities in marinated and unmarinated broiler meat by metagenomics',NULL,'HARVESTED','kaisa.koskinen@helsinki.fi ','K Koskinen','2016-01-20 14:12:06','Webin-52',31,'2014/02/ERP001021','2011-11-22 00:00:00','PRJEB2763');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1449,'NOFIMA',NULL,1,'2017-03-26','THe samples are to investigate the type of microbes that Atlantic salmon are feed through feed','ERP021211','Microbes present in feed to Atlantic salmon','FINISHED','SUBMITTED','christian.karlsen@nofima.no','Christian Karlsen','2017-02-10 13:11:22','Webin-42944',31,'2017/02/ERP021211','2017-02-10 13:11:22','PRJEB19227');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1636,'INRA',NULL,1,NULL,'This study is a large comprehensive survey of meat and seafood spoilage microbiota using 454-GS-FLX Titanium amplicon sequencing of the 16S rRNA gene V1-V3 fragment. Primer EBP-27F was used as a forward primer and EBP-534R as a reverse primer. Eight different products were analyzed including four meat products : Ground Beef (code sample = GB), Ground Veal (code sample = GV); Poultry Sausages (code sample = PS); Bacon Dices (code sample =  BD), and four seafood products : Salmon Fillets (code sample = SF); Smoked salmon (code sample = SS); Cod fillets (code sample = CF); Cooked Peeled Shrimps (code sample = CS). Ten samples were analyzed per food product (coded 01 to 10) and at two time of storage (T0 for the freshly produced food product; TS for the spoiled product obtained after a certain period of storage time at refrigerated temperature). Overall, 160 samples were analyzed (80 samples at each storage time of analysis). All samples were subject to pyrosequencing replication, one from the 27F primer A-side (EBP-XXXX-XX-F.fastQ files, where XXXX-XX denotes the sample\'s code) and the second from the 534R primer B-side (EBP-XXXX-XX-R.fastQ files). All samples from a given product at a given storage time and their associated replicates were pooled in a 1/4 lane of a 454 GS-FLX Titanium run yielding an average sequencing depth of 15,000 reads per sample (~7,500 reads/replicate). All FastQ files are corresponding to Quality-filtered reads (length > 250 bp).','ERP004317','Origin and ecological selection of core and food-specific bacterial communities associated with meat and seafood spoilage.','FINISHED','SUBMITTED','stephane.chaillou@jouy.inra.fr','Stephane CHAILLOU','2017-03-23 16:29:52','Webin-1533',31,'2017/03/ERP004317','2017-03-23 16:29:52','PRJEB4975');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1639,'UC DAVIS GENOME CENTER',NULL,1,NULL,'We characterized the microbiota of three different dietary patterns in order to estimate: the average total amount of daily microbes ingested via food and beverages, and their composition in three daily meal plans representing three different dietary patterns. The three dietary patterns analyzed were: 1) the Average American (AMERICAN): focused on convenience foods, 2) USDA recommended (USDA): emphasizing fruits and vegetables, lean meat, dairy, and whole grains, and 3) Vegan (VEGAN): excluding all animal products. Meals were prepared in a home kitchen or purchased at restaurants and blended, followed by microbial analysis including aerobic, anaerobic, yeast and mold plate counts as well as 16S rRNA PCR survey analysis.','ERP008907','The microbes we eat: abundance and taxonomy of microbes consumed in a day\'s worth of meals for three diet types (USDA recommended, vegan, and “typical American”)','FINISHED','SUBMITTED','jnamorgan@ucdavis.edu','Jenna Lang','2017-03-24 10:01:09','Webin-39496',31,'2017/03/ERP008907','2017-03-24 10:01:09','PRJEB7918');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1640,'INRA JOUY-EN-JOSAS',NULL,1,NULL,'Both salt concentration and packaging type can modify the nature and the dynamic of microbial ecosystems in meat products. The purpose of this work was to investigate the influence of these two parameters on the bacterial communities of raw pork sausages using a metagenomic approach and to correlate these data with sensory analysis.  Two concentrations of sodium chloride were tested: normal content (2 % w/w) vs. a 25 % reduction (1.5 % w/w) and for each, packaging was either vacuum or modified atmosphere (N2%-CO2%). Sausages were stored at 8 °C during 21 days until their use-by date. Spoilage assessment was performed by sensory and physicochemical analyses. To characterize the bacterial diversity, the 16S rDNA of five replicates per condition tested was amplified for pyrosequencing at a scale of 15,000 reads per library. Reads were then clustered into Operational Taxonomic Units (OTU).  Spoilage was characterized by an important greying of the products and a production of gas and off-odours defined as rancid, sulphurous and sour. It was more pronounced for samples under modified atmosphere and easier to be ascertained with the lowest salt content. The 20 samples taken altogether,provided 387,111 bacterial 16S rRNA sequences which were analyzed and binned into 78 OTUs (97% threshold).. Packaging atmosphere influenced the average bacterial richness with 69 OTUs (+/-?) identified in vacuum-packed meat and a lower diversity of 46 OTUs (+/-?)in modified atmosphere-packed meat. The richness was influenced mainly in the sub-dominant population whereas the core dominant bacterial species were similar in all samples and composed of Lactobacillus sakei, Lactococcus piscium, Carnobacterium divergens, Serratia proteamaculans and Brochothrix thermosphacta. However, salt reduction turned out to induce a significant drop in species diversity evenness among the core dominant species and thus led to a faster spoilage event..   This knowledge will help in defining strategies to offset salt reduction meanwhile keeping the safety of meat products.','ERP010603','Role of salt and storage packaging conditions defining the spoilage microbial diversity of raw pork sausages','FINISHED','SUBMITTED','lfougy@jouy.inra.fr','Lysiane Fougy','2017-03-24 10:22:01','Webin-40756',31,'2017/03/ERP010603','2017-03-24 10:22:01','PRJEB9493');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1754,'BioProject',NULL,1,NULL,'Exploring the sources of beef microbial contamination and study the beef microbiota evolution after aerobic storage at 4°C.','SRP021108','Beef metagemome','FINISHED','HARVESTED',NULL,NULL,'2017-05-17 12:06:57','Webin-842',31,'2017/05/SRP021108','2017-05-17 12:06:57','PRJNA196434');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1773,'BioProject',NULL,1,NULL,'Dietary supplements with live microbials.','SRP062833','Dietary supplements Metagenome','FINISHED','HARVESTED',NULL,NULL,'2017-05-24 15:20:47','Webin-842',31,'2017/05/SRP062833','2017-05-24 15:20:47','PRJNA291686');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1797,'BioProject',NULL,1,NULL,'metagenome of Lavandin honey microbiome','SRP097100','Lavandin honey microbiome Metagenome','FINISHED','HARVESTED',NULL,NULL,'2017-06-12 09:42:28','Webin-842',31,'2017/05/SRP097100','2017-06-12 09:42:28','PRJNA360677');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1798,'Institute for Genome Sciences - UMB',NULL,1,NULL,'Tomato (Solanum lycopersicum) consumption has been one of the most common causes of produce-associated salmonellosis in the United States. Contamination may originate from animal waste, insects, soil or water. Current guidelines for fresh tomato production recommend the use of potable water for applications coming in direct contact with the fruit, but due to high demand, water from other sources is frequently used. We sought to describe the overall bacterial diversity on the surface of tomato fruit and the effect of two different water sources (ground and surface water) when used for direct crop applications by generating a 454-pyrosequencing 16S rRNA dataset of these different environments. This study represents the first in depth characterization of bacterial communities in the tomato phyllosphere and the water sources commonly used in commercial vegetable production.    The two water sources tested had a significantly different bacterial composition. Proteobacteria was predominant in groundwater samples, whereas in the significantly more diverse surface water, abundant phyla also included Firmicutes, Actinobacteria and Verrucomicrobia. The phyllosphere bacterial communities on tomatoes sprayed with both water sources could not be differentiated using various statistical methods. Both phyllosphere environments had a high representation of Gammaproteobacteria, and within this class the genera Pantoea and Enterobacter were the most abundant.    Despite the major differences observed in the bacterial composition of ground and surface water, the season long use of these very different water sources did not have a significant impact on the bacterial composition of the tomato phyllosphere. This study has provided the first next-generation sequencing database describing the bacterial communities living in the phyllosphere of a tomato crop under two different spray water regimes, and therefore represents an important step forward towards the development of science-based metrics for Good Agricultural Practices.','SRP005615','Bacterial community diversity and variation in spray water sources and the tomato phyllosphere','FINISHED','HARVESTED',NULL,NULL,'2017-06-12 09:47:40','Webin-842',31,'2017/05/SRP005615','2017-06-12 09:47:40',NULL);
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1804,'BioProject',NULL,1,NULL,'Microbiome of wild lavender honey','SRP097098','Microbiome of wild lavender honey Metagenome','FINISHED','HARVESTED',NULL,NULL,'2017-06-13 15:20:09','Webin-842',31,'2017/05/SRP097098','2017-06-13 15:20:09','PRJNA360674');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1838,'BioProject',NULL,1,NULL,'We describe here the metagenomics-derived viral sequences found in beef, pork, chicken purchased from supermarkets in San Francisco. Meat and meat products can be the source of numerous enteric infections in humans. A library was then constructed using Nextera XT DNA Sample Preparation Kit (Illumina) and then sequenced using the Miseq Illumina platform with 250 bases paired ends with a distinct molecular tag for each pool.','SRP044165','Viral metagenomics of US store bought beef, pork, chicken','FINISHED','HARVESTED',NULL,NULL,'2017-06-20 14:32:24','Webin-842',31,'2017/05/SRP044165','2017-06-20 14:32:24','PRJNA254745');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1842,'BioProject',NULL,1,NULL,'fresh bagged spinach spiked with STEC','SRP083779','food contamination metagenome Metagenome','FINISHED','HARVESTED',NULL,NULL,'2017-06-20 14:36:31','Webin-842',31,'2017/05/SRP083779','2017-06-20 14:36:31','PRJNA340876');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (830,'BioProject',NULL,1,NULL,'investigate the bacterial and archael community composition in landfill and assess the variation of the bacterial and archael compositions in refuse decomposition process.','SRP029903','Shanghai Laogang landfill leachate','FINISHED','HARVESTED',NULL,NULL,'2016-05-04 10:09:37',NULL,62,'2016/05/SRP029903','2016-05-04 10:09:37','PRJNA219026');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (518,'GSTT BRC Bioinformatics',NULL,1,'2017-02-03','These samples are selections from a larger cohort that were selected for the participation in the EBI metagenomics training in Sept. 2015','ERP011339','These samples are selections from a larger cohort that were selected for the participation in the EBI metagenomics training in Sept. 2015','FINISHED','SUBMITTED','filipe.gracio@kcl.ac.uk','Filipe Gracio','2016-01-20 14:12:06','Webin-41270',401,'2015/09/ERP011339','2015-08-04 00:00:00','PRJEB10133');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (520,'LEIDEN UNIVERSITY MEDICAL CENTER',NULL,1,'2016-01-31','Whole genome sequencing of metagenomes extracted from palms of two individuals','ERP011525','Whole genome sequencing of metagenomes extracted from palms of two individuals','FINISHED','SUBMITTED','l.khachatryan@lumc.nl','Lusine Khachatryan','2016-02-01 00:05:03','Webin-41345',401,'2015/09/ERP011525','2015-08-12 00:00:00','PRJEB10295');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (604,'NIAMS',NULL,1,NULL,'16S rRNA gene sequences amplified from subjects with eczema and age-matched healthy controls.  Microbes living in and on humans are ten times more numerous than human cells. Culture-based methods have been the primary techniques used to study microbes inhabiting humans; however, many species are not successfully grown in culture. The NIH Roadmap for Medical Research Human Microbiome Project (HMP) aims to investigate the microbes in the gastrointestinal tract, oral cavity, skin, vagina, and nares. The goal of the HMP is to comprehensively characterize the human microbiota and analyze its role in human health and disease.   The skin serves not only as a barrier against invading pathogens and moisture loss, but also as a host to microbial communities. The skin of an adult is an approximately 2 square meter surface with a myriad of microenvironments.  Atopic dermatitis (AD), more commonly known as eczema, is a common skin disorder that is exacerbated by Staphylococcus aureus colonization.  This study aims to investigate the skin microbiota of AD patients at specific timepoints (quiescence, disease flares, and post-treatment) to examine how disease state correlates with changes in the skin microflora.     The 16S small subunit ribosomal (rRNA) gene is present in every bacterial cell, serving as a universal marker. The gene is sufficiently conserved to allow accurate alignment but adequately varied to enable phylogenetic analyses. Sequencing via 16S rRNA-based phylotyping has been used to survey the bacterial microbes on the skin.  Initially, we will examine the bacterial diversity associated with AD using a 16S rRNA survey and later broadening to include fungi, viruses, archaea and mites. In addition, bacteria will be cultured on a wide variety of media and isolates representing both abundant and novel species will be selected for whole genome sequencing.  To characterize fungal diversity, we utilize two phylogenetic markers within the rDNA region: 18S rRNA and the Intervening Internal Transcribed Spacer (ITS) region.  Full microbial diversity will be explored with shotgun metagenomic sequencing.','SRP002480','Gene-Environment Interactions at the Skin Surface','FINISHED','HARVESTED',NULL,NULL,'2016-02-03 11:24:47',NULL,401,'2016/01/SRP002480','2016-02-03 11:24:47','PRJNA46333');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (605,'BioProject','Infection',1,NULL,'The aim of this project was to investigate the interaction of the skin microbiome and Haemophilus ducreyi during experimental infection of human volunteers. We hypothesized that the skin microbiome influences susceptibility to H. ducreyi infection and that infection with H. ducreyi causes changes in the skin microbiome. To address these hypotheses, we collected skin swab samples prior to inoculation with H. ducreyi, at Days 1, 2 and 5 post-inoculation, at clinical endpoint, and at the test of cure visit. Samples from four dose-matched pairs of volunteers that either resolved the infection or formed pustules were subjected to barcoded amplification and pyrosequencing of the V1-V3 region of bacterial 16S rRNA genes. We compared alpha and beta diversity measurements of the microbiomes of resolvers and pustule formers, and determined signature bacteria that were associated with infection resolution or pustule formation.','SRP056364','Skin microbiome in human volunteers inoculated with H. ducreyi Raw sequence reads','FINISHED','HARVESTED',NULL,NULL,'2016-02-04 10:26:33',NULL,401,'2016/02/SRP056364','2016-02-04 10:26:33','PRJNA278873');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1187,'Institute of tropical medicine',NULL,1,'2016-10-30','Background: Buruli ulcer (BU) is an infectious disease caused by Mycobacterium ulcerans and considered the third most prevalent mycobacterial disease in humans. This skin disease can affect the entire body surface, but primarily the lower extremities, with symptoms varying from nodules to ulcers. Secondary bacterial infections in open BU lesions are the main cause of pain, delayed healing and systemic illness, resulting in prolonged hospital stay. Thus understanding the diversity of bacteria in these open lesions is important for proper treatment. The normal skin flora is known to protect the host through commensal relationships, however, certain factors such as ulceration can shift the skin flora from being primarily commensal to potentially pathogenic. Therefore, the objective of this study was to characterize and compare the skin bacteria from BU ulcers, non-BU ulcers, and similar locations on the skin of healthy individuals. Results: Using 16S rRNA sequencing, we determined the microbial composition of 5 BU lesions, 3 non-BU lesions and 3 healthy skin samples, and compared these using a data analysis pipeline. Our results showed a lower bacterial diversity in both the BU and non-BU lesions compared to the healthy skin. However, no significant differences were found between BU and non-BU lesions. The BU lesions were characterized by an increase of Bacteroidetes compared to the non-BU wounds, which contained more (Gamma)Proteobacteria. Furthermore, the BU lesions also contained significantly more obligate anaerobes. All lesions contained a mixture of gram-positive and gram-negative bacteria, with more gram-negatives present in the BU and non-BU lesions than in healthy skin samples. With this molecular-based study, we were also able to detect bacteria, which were missed by culture-based methods in previous BU studies.Conclusions: Our study suggests that BU leads to changes in the bacterial community within the lesions. These changes are potentially detrimental and may cause a delay in the healing process. In order to determine if the alterations in Bacteroidetes were due to a specific wound environment, underlying pathophysiological conditions created by M. ulcerans, or associated with wound location on the body, further microbiome studies are necessary.','ERP016629','Microbiome samples derived from Buruli ulcer wounds and non-Buruli ulcer skin ulcerations','FINISHED','SUBMITTED','cmeehan@itg.be','Conor Meehan','2016-07-29 14:04:21','Webin-44249',401,'2016/07/ERP016629','2016-07-29 14:04:21','PRJEB14948');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1295,'Universidade Federal do ABC',NULL,1,'2016-12-23','Using high-throughput 16S rDNA and ITS1 sequencing, we characterized skin bacterial and fungal microbiotas from healthy and dandruff subjects, comparing scalp and forehead (lesional and non-lesional skin sites).','ERP018577','Human skin bacterial and fungal microbiotas analyses','FINISHED','SUBMITTED','pennaphc@gmail.com','Pedro Penna','2016-11-03 10:19:27','Webin-44666',401,'2016/11/ERP018577','2016-11-03 10:19:27','PRJEB16723');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (1818,'LSHTM',NULL,1,NULL,'Microgravity induces physiological deconditioning due to the absence of gravity loading, resulting in bone mineral density loss, atrophy of lower limb skeletal and postural muscles, and lengthening of the spine. SkinSuit is a lightweight compression suit designed to provide head-to-foot (axial) loading to counteract spinal elongation during spaceflight. As synthetic garments may impact negatively on the skin microbiome, we used 16S ribosomal RNA (rRNA) gene amplicon procedures to define bacterial skin communities at sebaceous and moist body sites of five healthy male volunteers undergoing SkinSuit evaluation. Each volunteer displayed a diverse, distinct bacterial population at each skin site. Short (8 h) periods of dry hyper-buoyancy flotation wearing either gym kit or SkinSuit elicited changes in the composition of the skin microbiota at the genus level but had little or no impact on community structure at the phylum level or the richness and diversity of the bacterial population. We also determined the composition of the skin microbiota of an astronaut during pre-flight training, during an eight-day visit to the International Space Station involving two 8 h periods of SkinSuit wear, and for one month after return. Changes in composition of bacterial skin communities at five body sites were strongly linked to changes in geographical location. A distinct ISS microbiome signature was found which reversed to a pre-flight profile on return. No changes in microbiome complexity or diversity were noted, with little evidence for colonisation by potentially pathogenic bacteria. We conclude that short periods of SkinSuit wear do not compromise the healthy skin microbiome.','ERP022958','Impact of the Mk VI SkinSuit on skin microbiota of terrestrial volunteers and an International Space Station-bound astronaut','FINISHED','SUBMITTED','richard.stabler@lshtm.ac.uk','Richard Stabler','2017-06-16 11:39:52','Webin-40437',401,'2017/06/ERP022958','2017-06-16 11:39:52','PRJEB20777');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (2062,'EMG',NULL,1,NULL,'The human wound Third Party Annotation (TPA) assembly was derived from the primary whole genome shotgun (WGS) data set PRJNA344941. This project includes samples from the following biomes : Human skin.','ERP104236','EMG produced TPA metagenomics assembly of the Identification of fungi and ameba from human wound genomic sequencing (human wound) data set','FINISHED','SUBMITTED',NULL,NULL,'2017-11-24 16:47:21','Webin-460',401,'2017/11/ERP104236','2017-11-24 16:47:21','PRJEB22555');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (2072,'The ithree institute',NULL,1,NULL,'Background: Type II diabetes is a chronic health condition which is associated with skin conditions including chronic foot ulcers and an increased incidence of skin infections.  The skin microbiome is thought to play important roles in skin defence and immune functioning.  Diabetes affects the skin environment, and this may perturb skin microbiome with possible implications for skin infections and wound healing.  This study examines the skin and wound microbiome in type II diabetes. Results: Ten type II diabetic subjects with chronic foot ulcers were followed over a time course of 10 weeks, sampling from both foot skin (swabs) and wounds (swabs and debrided tissue) every two weeks. A control group of 8 non-diabetic subjects was also followed over 10 weeks, and skin swabs collected from the foot skin every two weeks.  The diabetic skin microbiome was significantly less diverse than non-diabetic skin.  Community composition was also significantly different between diabetic and non-diabetic skin, however the most abundant taxa were similar between groups, with differences driven by very low abundant members of the skin communities. Chronic wounds were generally colonized by the most dominant skin Staphylococcus, along with other taxa that generally different by patient. No significant correlations were found between wound duration or healing status and the abundance of any particular taxa, while wound size was positively correlated with the dominant skin Staphylococcus.   Conclusions: The major difference observed in this study of the skin microbiome associated with diabetes was a significant reduction in diversity.  The long-term effects of reduced diversity are not yet well understood, but are often associated with disease conditions.','ERP019566','Longitudinal study of the diabetic skin and wound microbiome','FINISHED','SUBMITTED','catherine.burke@uts.edu.au','Catherine Burke','2017-11-27 14:44:28','Webin-41359',401,'2017/11/ERP019566','2017-11-27 14:44:28','PRJEB17696');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (2217, 'BioProject', NULL, '1', NULL, 'Oral Microbiome Characterization', 'SRP062418', 'Oral Microbiome Metagenome', 'FINISHED', 'HARVESTED', NULL, NULL, '2018-02-14 14:15:20', 'Webin-842', '363', '2018/02/01', '2018-02-14 14:15:20', 'PRJNA292800');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (462,'OSD-CONSORTIUM',NULL,1,NULL,'Sequencing of amplicon and metagenome samples from the main OSD event, representing joint effort of marine sampling stations around the world. The OSD campaign aims to analyze marine microbial community compositions and embedded functional traits on a single day, the solstice on 2014-06-21.','ERP009703','Ocean Sampling Day (OSD) 2014: amplicon and metagenome sequencing study from the June solstice in the year 2014','FINISHED','SUBMITTED','rkottman@mpi-bremen.de','Renzo Kottmann','2016-01-20 14:12:06','Webin-31235',132,'2015/03/ERP009703','2015-03-01 00:00:00','PRJEB8682');
INSERT INTO `STUDY` (`STUDY_ID`,`CENTRE_NAME`,`EXPERIMENTAL_FACTOR`,`IS_PUBLIC`,`PUBLIC_RELEASE_DATE`,`STUDY_ABSTRACT`,`EXT_STUDY_ID`,`STUDY_NAME`,`STUDY_STATUS`,`DATA_ORIGINATION`,`AUTHOR_EMAIL`,`AUTHOR_NAME`,`LAST_UPDATE`,`SUBMISSION_ACCOUNT_ID`,`BIOME_ID`,`RESULT_DIRECTORY`,`FIRST_CREATED`,`PROJECT_ID`) VALUES (573,'UNIVERSITY OF WARWICK',NULL,1,NULL,'The final aim of this study is to identify different soil environments potentially rich in novel natural compounds correlating microbial community diversity and biosynthetic genes for polyketide synthesis. The identification of interesting environments for novel natural products will be useful for further drug discovery pipeline development.','ERP013097','metagenomic amplicon sequensing from soils for novel natural products (16S_PKS)','FINISHED','SUBMITTED','chiara.borsetto@warwick.ac.uk','Chiara Borsetto','2016-02-03 10:02:09','Webin-41944',220,'2015/11/ERP013097','2015-11-06 00:00:00','PRJEB11689');
